Description: regenerate SSL certificates using the updated generate.sh script
Author: Santiago Vila <sanvila@debian.org>
Forwarded: not-needed
Last-Update: 2023-12-09

--- a/examples/remoteobjects/ssl/sslserver/cert/client.crt
+++ b/examples/remoteobjects/ssl/sslserver/cert/client.crt
@@ -1,17 +1,17 @@
 -----BEGIN CERTIFICATE-----
-MIICrTCCAZUCFHOQggvUf1o8c5i3yNyiGLNcLC4rMA0GCSqGSIb3DQEBCwUAMBIx
-EDAOBgNVBAMMB1F0Uk8gQ0EwHhcNMjMwNTMxMDgxMTE1WhcNMjUwOTAyMDgxMTE1
+MIICrTCCAZUCFHOQggvUf1o8c5i3yNyiGLNcLC4vMA0GCSqGSIb3DQEBCwUAMBIx
+EDAOBgNVBAMMB1F0Uk8gQ0EwHhcNMjMxMjAzMjE0NDM4WhcNMzMxMTMwMjE0NDM4
 WjAUMRIwEAYDVQQDDAkxMjcuMC4wLjEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAw
-ggEKAoIBAQDIP/hpIFwU9K1PolZWlgI3k8s8+ZqetECXlL6zf1ZLUWsDhPDeZDLT
-xXENiBnEnwomOXesB0G4lcD3r/oWZsEQK3mtukeZEsHRe/bllkqf3LuBcAOt62da
-it7On6cBSZNJLL6Hu1ctXi5L/mGoglDu9/2C//zJzj8KKjrDfbxMIuw2/guttYpM
-TzPNyrcA0OVaMAcw0niIbZlpJOn1q+sAuDXs1cAYYuNjkj6EUb7WnXJHDKUWu+q7
-WoBISfqH0p5bjEFz7VZOKJW25BXbJ6kpHMowBuyvNs+cTfWdRVNfVT+YJz0Vmyvb
-cUfFFEzmIIFJs9pH8nmHu6rpAgFfFOyhAgMBAAEwDQYJKoZIhvcNAQELBQADggEB
-AF0FzC0gRY0s6rAoJ+WV5WwCLahvUi4hR+Uug09rn1dgccDXMOcsFqGa9IKGaumq
-pWL/5aZzwt8izJ2LU3wYPE57bP21wRrVwvdIWX9CKab5DA8kL1BzQsoa+isS/beR
-dQjB2L9KZWRDkRVEJcyY/vfBkdJmwHTPBFx/EJjIwA93qfEvuif+bGL7xUVjIgxi
-MsSox3BJSdyNQ6MyoumCmUcD+iTCLQsavWpQuUv8nhXoTA2o2ozFSrDvWEuxwZyJ
-VRObyZLs1jJPQXAbKkv1DFFdlt3GlbS1TTjnlFMrZ77caDFX/CX01z37qqFTZw88
-S3FnpcsSk029XFie37tCYb8=
+ggEKAoIBAQC+b2yTU2pDQoS+fYKmOLVh/KkUgAU6ZV3g2ywmEdxJzF6knJqvwFui
+4MxfR9XACctEW+xxtklPy4aRFFWptYHi9v8V+Ac6SVEukGYP0A52Jn9/MqIZwu7e
+dwoWF1p5GVKkjz09FsnaOrWw8RRuD5jqcAheaURYRHiqveZxzxiBpmvgJpu2K1kf
+QS22fbnZGfRYdBcjhX+txOo3wt19qcgIofqh4M9N6gCwXt75VEEOuWwI8FCgX2y3
+n0p2nHM7iR4wPBYXJpV25K6FEM5AvQYkSjYpNwaggNTMv63xgtGEAPWC0XxA4Qfq
+Nxnj4qn4SU1lHr6nxVlDRox7yCUa58vbAgMBAAEwDQYJKoZIhvcNAQELBQADggEB
+AElzoUkYaV1uWPFZceYvEG2IOIl33yEMYqcRbdGRHimBA6hiupencxv+UG51aZ7Q
+un6sI01Nj4weeSQGWurl13qPoUJ5RvY295bDR2kS+qnb+9wG8H9/q/pq1yrlH5ZW
+IxWqJlPfUBKj/ShEplrDCn0OR38+tvNKAPi9PQVi9D8HD1SOtuhcLk4hU/Hqj9ek
+X/8kueT2bIc5u6rAoYktrQ3g0fqyfZUVezIqd0YlI9jUNRCfRd10Fyu+c8ODgCCS
+fXZ0HnBMWxfkF557O9q6MO2tWOJv+qq9qw9t5I/kduezkdGhMTeXv3NhxIHIYUTG
+CSLQbje++hKe5M7y5z6BoQ4=
 -----END CERTIFICATE-----
--- a/examples/remoteobjects/ssl/sslserver/cert/client.key
+++ b/examples/remoteobjects/ssl/sslserver/cert/client.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDIP/hpIFwU9K1P
-olZWlgI3k8s8+ZqetECXlL6zf1ZLUWsDhPDeZDLTxXENiBnEnwomOXesB0G4lcD3
-r/oWZsEQK3mtukeZEsHRe/bllkqf3LuBcAOt62dait7On6cBSZNJLL6Hu1ctXi5L
-/mGoglDu9/2C//zJzj8KKjrDfbxMIuw2/guttYpMTzPNyrcA0OVaMAcw0niIbZlp
-JOn1q+sAuDXs1cAYYuNjkj6EUb7WnXJHDKUWu+q7WoBISfqH0p5bjEFz7VZOKJW2
-5BXbJ6kpHMowBuyvNs+cTfWdRVNfVT+YJz0VmyvbcUfFFEzmIIFJs9pH8nmHu6rp
-AgFfFOyhAgMBAAECggEAAUU0nngb1W8y7UMmoJB9jZw0qRudQT37/eot5oYbhklL
-MD/G5mZE00aYn8EaBr8KibEKIw99uX3LtlaJs9wtg9x+vl0k6KHvlFRgvdac/RQh
-pWHQSClq465nWPC21sGAe3Ga+NH1rN5A1W8U8AR5nnZ4x3y3vhD+oeIgMs7Io8P/
-EiQIJoOWPFBtDR+7zSZ3W8zjXSHezI0fclhtXZIpwSiOnOSUUhzz73ObEU66tKa2
-LqA7WiQGWfV/wqMPOEeVBoD19HrgsinCvmpXtwsy69PpIk07Vdn7zbYkfHF49Uud
-/lfH3t4vAcnvJFlfPhgXuZzUBp7MSbk5y7EULnnwFQKBgQDxt6kOtwPIkpsHJz2i
-4Mlq053lnaH78xzwf3SH0HvcryiBps8Je5dBZ9xBwFHuwbn1cA7KzIs54Z8pBC3D
-fPDkT/X8a1pdpt9xZLvvmZ1GX2RGFR+gs1Ii5yNIrL5nsqYAoUaVGYJWBTXMskFO
-YHNVtmz418//orp7FsNyis2WbQKBgQDUFQ0Sy3IGwDlCohC9/bCm2Yu39d+2DPk5
-U60susxpQJcUhBlcv2rGhonLjwP2BilTEYqQ8IUN1uYnnYFw3lyJMZ5MxFQOVWrB
-/+Z9w10mRg2DgzPftrt6P+YNGTE8ygpai0JuAyUS0EPWB+LLIKPkLN/23gBbtZVD
-k4PBIhkehQKBgQDAG02bSk0RfaMEfmV4K9soseMN+Mgiq8dWRJXi0wl+DR/OWzq0
-uKsv6lGdWDJF5WXb8SFwgDwu1mheetm8QnJZoktIQDHjuJphTsQm2xmqMmpJYMRw
-PdSLtNPQEKqLU+qnGYY1Pb7eiumFNC6/EdzYRFI7O5ot+7Dl3uObB0H5yQKBgGMp
-sKIwFBHKATwBsRhUrStmnqiWQ1LJACLo2BkJApjYBYwnNACk09UdhQJzzWNUgDAC
-DKs2es/8wkKVp7dRkSzo0V6Nkcvd7nvlHi2BkmVZ7emnCDM7Kq+/ft5zAA2vXmX1
-pUrxF32TeZRyvX1Ctr7yqImyRbKb1b3yFHHScrJNAoGAc2eHAbmW3MjPwbHcdd8E
-ytL8wZhLmCBrAjUwv2qkAieh8meydLwNipl2gI95E1FmM4J5uFH8kB0WskcZ9vcC
-BK7He8JG9DoW8nHvupMxbHV3U30A65mfBn0C3/Oj+aHQtclXOo0odBSOzmghBn5F
-suxfBRpfio6OvYi192d7QCU=
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC+b2yTU2pDQoS+
+fYKmOLVh/KkUgAU6ZV3g2ywmEdxJzF6knJqvwFui4MxfR9XACctEW+xxtklPy4aR
+FFWptYHi9v8V+Ac6SVEukGYP0A52Jn9/MqIZwu7edwoWF1p5GVKkjz09FsnaOrWw
+8RRuD5jqcAheaURYRHiqveZxzxiBpmvgJpu2K1kfQS22fbnZGfRYdBcjhX+txOo3
+wt19qcgIofqh4M9N6gCwXt75VEEOuWwI8FCgX2y3n0p2nHM7iR4wPBYXJpV25K6F
+EM5AvQYkSjYpNwaggNTMv63xgtGEAPWC0XxA4QfqNxnj4qn4SU1lHr6nxVlDRox7
+yCUa58vbAgMBAAECggEAGoCGMzCSIr8gmyP7aSeOMo9Hdodkct6FhJPr3nZw2kJb
+T3xN2LSj7BRtzLrCCAI07BQA4UhVhonT9d/HFiGg7FWkZUnJs7ZdWmYVFhFF2fJ8
+ntaMkZo7xPD7Edz5Jc483pLimlxQBhNSksfSKIj6JnIl8AGlVcVdp6Fa7y67gdtK
+L6tJ+u3azUi9yry4VzNOLgQNZLaMQ//vuBjql8C/1Owq2AEdOlweTKZQ0NzKV3D4
+GL/nyakwX6dYqNz6//VXeaYK9hCOMFyO+w0GYTGrG8ryGSVoYMNRKib/DXCiFrHA
+9Vz9btZWp7M81NkuEymYdqcNJwciDwG7UYszqlZkxQKBgQDo99025fO8proFPVB3
+DbbfIb4mJuLi8OBweS7PTBg+8RugppAWLARrNN9EhmgNKu8SKCL02Tq3keqqx0uy
+L9zBIykpzA+LxPbvhXdNu98mTNNziwlLKO4kqhuC0j3FSgGhRLC7f4nZ6WWNEeGr
+6GChdnYO/0wl/WMcrZuwYVk1pwKBgQDRQxp8psrlGDH3t/57ezgLAHKQrQjBLwOO
+MQY/UrGkxtsG75HdLFtQNy0Yte79Y1byIVwoGgfduZQ5SNjSFcEfQhvuEpn09/sn
+PgYr9MLOxup56uPa3wAdmpaMpf50T1V+2PvrKGMNjgVV+MfG09ZTQahQDdluYbJA
+WPyNC5JmrQKBgQCgyfGz4iHHIotg8mCGysBcYD2Mt2oAl59QAVN9X1ecmWwoBM2N
+f/PMykEOWnWWIPu/3FPECemsqNj95W8pHD+QTWQrpNtV7CtCRSuCSxhIpt2J8XwI
+k8dJJt61+8UGVX2qgG45+lx6OcwNATpwG4IV7lytHSbBUZEGSgwaQBYCIQKBgE53
+OPFQoXex0NqwNU4b7/HAIzU4jLXTaSPiIhOQcGf24J4DKtCiF0NGzTQc1BZTL4Dv
+HV00FYsAUSIBvLtZyOZtpYdpli0S5yvZx/qndnyOfdFlUDb7s1Fma9klTPpT3R0J
+Q7zzkeC9RibseAyH1i+1yOTrrIIFRA94zoocsrKpAoGACywVD7XjcYG5GTvJ90L+
+UH5CnAD8vZZtZC2eYq37JxYgWs8yGooszlXfVXcYuQCZ8lPH8EEq+jD0QW/TJeAn
+q6LZVtmOsqTwhEaxxpJH/eh0SghfHDOUgaBxx7ufor0He4hciRnQqh43Dx1DMfRh
+23qWSBduVOdEUO+K9tHAMNg=
 -----END PRIVATE KEY-----
--- a/examples/remoteobjects/ssl/sslserver/cert/rootCA.key
+++ b/examples/remoteobjects/ssl/sslserver/cert/rootCA.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDR8adUoBxw9L7E
-5rkMbvNmY2i5aJUecLSogXZ5arctuBeu4PQC2YzZU/0ff/kq8n/NhXATH11D7CXa
-d7pYPbMaP1dqPkBdTAkRjkpwlkhGHQv/9teQvMWesVr8rQ4ddk2DK/jmEEyFD3Mw
-TgX8jsw7nhM+mBnhbvK/7DEEoAK6pfDuLl8fsVAHdc+3RClO0FQAYt1psvPB8Yvq
-x2NXy3GKRQn+wkr2pVysDkaEEqtgkE2KlD1Qfyqr5pnN1wDRtkO4syLLlQ9WECRa
-qU848FFWIH9GkJkx7nmRTEd0Ma/WS0O7BnXGxD3fRxNKtc7+PVxvdDb/roq2xiq9
-6ww3f4IvAgMBAAECggEAVsf/FPoCB6cDXDzK7RrHveEFnu5eBXEtHCI3qMS1504b
-bAxMkQkAMa2ICgdKJbiLn5ay1xS15qeslfu7qVIHwKfCKIaG8InHjdw1ZQiZ43GK
-TknonUmKnqhNJfkaxo7qR3hbsC24k5/pS2RVESqtvU8AQXV57TeBXFfzgyfpOOEz
-Nz4c27HfV5AZsDB4GOW1CS7ZdmvZuAfThhTJKh0y1Lqn/7TcVLEStDnFyUZig9gr
-I7WaxltruTgdmmhl5qX0Y1vgwS7APGGLYWTmRRBvyloyqfggCYeU3HDF3qH6RStK
-PsPbbHYW6Xj6LxZSJfVLdAVfFim0yAH1fTAmQXiOkQKBgQDdzLSA5zUBAIR3CTms
-sCn8a0ud8ETPUmTf1RmBMDeIuBTR3va/Q0YVYF8jY+ANrEwut0quIgOwZT4Kmo36
-Sx3F+uE7Gy0XQtDnZk7HeCOsF1fY7r81z7oAh61vFxG8ejSMqjUH8S3tTaCBvGi5
-z2ZjDX2lOsynkBwHu+jvYCbwRQKBgQDyUPWHkO41f6SoTXrEzf77y7Z5Wlim2Ltq
-gLAaEjLPuPsL+G/v7nCw6k6YmRli9f5jn1MGCzaRTxu1hkZlPhPvmfH/AGoSixgB
-CGKMCzOuFT3lQmpr0PZdJBpQ0aAo0R8X48J1X1WuyhZFRmafsrqcYeYPLLrs8aSp
-F51EiOJx4wKBgHZsxjPNP0+5pwCnvFOn+8L6xvUdRKAtrg6VCXkg9YXaQ8OK+NRG
-ZzUir6mFuiuSw1puI0G2h6G3bNjcjNIHKW579aZJbFnvIvtqdMWsL9/C1vTNBOsG
-bpm517Bo8N9hoI06k/YZoBwERp7+qiwLwO9CVMPUEVGmcFDw0/57qE7BAoGAXNwv
-q5sb3n6jAChpAuCada4UqJ652H64G9FjKyqMNFSESIsVgpISt4mjQxFwzB7PNShK
-bceASuFREhh6A7RteWW6emmTPLX3JfYrpdVpIa5cQO5Z0CWNNBZwy0X6JYo2azZK
-QCaftFMXg8a97zdUEjfmLSyMGd9kpbQ5/ZijhtcCgYBpmEOKN1SuGyEQ8gTz/02j
-Xz0XW63Dw8JX8s1Ck5u2uM96gsnHfwuC+iZTSu6FIS1e+3NOCEGQVy10rMzR/TLq
-h13ryz65qQtjaNl1gkD2Xf/ENxML57CU9nK5v+0zb+Qlk3CMKm0N06UmUTUDcE/T
-wIMUuL1LfdQ0b4bnS5JjjA==
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCXDWVfqxLFohyp
+yfp/DS/tphmvt61c9WUAH7o34CmYXTjZlxkdtmhFHQznRW12oyt1Y78PQq0sbngL
+/uukb+3axYRIN4wQSLkAIVvdm6UbP7TnxzkCrGuEB0IEQuMKEGWYgKHwgwVcPovJ
+jsqYPZVvyoixlBInz9FgdjbhlSPsFOSTVvZzK5Vzv6mO9Jff60P2ZoUcoiDrJLUz
+nM6ITQ7+c38dZerOuBFV73N0CIsp4mO25//kqg0JXpmkAgt8TprOV3ngcuTCrdCe
+1SyLY850DO1cQj81HvSzlG6tkhfKKJx2NfKEMGhYl/DKc6er8q1ld2eAdNnMJ5Yr
+3UqQJRvjAgMBAAECggEABCdyfe1hfcfrNowpycFeFmYMAiDGBsQaO/pOD71Sx4WW
+IP3JILUPLF9pJxm/XJTjhr3wgjCY50Y+d5ineFqPqwWQPBWIUjuobXrStkcn2kD9
+Q6aJ8ys0uxPMBVFQQcruUFcGnRZYA7prSRFr5utCpyllwE8h1LB1bKFeTwPf/LJn
+lFHLr34Vmr6whW3dSOn/QyjSRWXx6b88xc32yfniqI8JZrC5ul0J1ulGB0QRhGkB
+tOr2udYIZKjJkgYPQAL644NDEXL5B5iWUvzVzAyNTsLD/uSb4M/KwWxBlM1kvOGU
+GSNr/yKhwqHvV7zpAyjttEltRi44Nt9ULekEVUB+WQKBgQDHgC+vzZRisjhF46rn
+h5OkPMIWEWOAbJ0+fWFDGncDaTOc++QM1WNgJR24zODTKl92EwQHDVAMesPpO614
+DaoKarmvSqcSRDjKvcyij0GTnDe8qPYPn5aJd9vW4vSsqGNUlevClosCxiavSDrV
+Dyt48PejuX+UW3iqkl1QdTiKNwKBgQDB1LK1lcMrQJBY55lpsF2ovvsEDtwOPR93
+Icr3VtMpZTB0R442WzpvAhKaDLLaKUp5pzRGL4tN3pfpD2tz15+neKn4saE8zLoo
+1V0VDoBx9PCvfoP98/zbXZJiVFkdur+B6kz2ImUdnajL2lgnZjJSKGU3+QcEa8be
+vCEbuuI1tQKBgEtK57Rz5BKzrGVEHVYwZtpPS/jEK0dmtpnE/BY1QMF6zM3siw8E
+4fKBT4xVVmyby1dmqdh3f8SV+KZnrXpnvcscqGOtywwrzztFEpwvcNJi3O5I3kTb
+OBc9syyQj0MycfqUclwIMQKlAiiZHAsZJJEKcIvMEIhPVWUaxDj/ZI1VAoGBALuz
+JXwCkwUHeMEfRci7KXeHubysRxziw92xs8LhKYskmk+AG483uNZWw3y6X1g0NBKq
+gEyXLecybxkZ/YwoekrK5o4WNXWTK5LvSzxg1rGD6K+JbHg4Fdy4OFFQLRxFA77j
+DbQsio6rZhJCVquQo5o71Ajn9oupcuOuesFHsFchAoGAYedyPqf8UDgTbsPlA2uo
+kzPanG7Vb5JtHbE14XsAaQP27EX6Ww96oCjEMEhq3KqHj+CQGga0uj6dhgWw+8fl
+s8sLz456U6XkRNjjDJ5luklxvNY2ejETcnZ16t0dp8TtJ8xiFvYO3XyIioCMdtY9
+3TVra0Izy2Wgc1dg4Pz9VKA=
 -----END PRIVATE KEY-----
--- a/examples/remoteobjects/ssl/sslserver/cert/rootCA.pem
+++ b/examples/remoteobjects/ssl/sslserver/cert/rootCA.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDBTCCAe2gAwIBAgIURtkt3RNfea7kp8Dl9UBvr0mXk0kwDQYJKoZIhvcNAQEL
-BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzA1MzEwODExMTRaFw0yNTA5MTMw
-ODExMTRaMBIxEDAOBgNVBAMMB1F0Uk8gQ0EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
-DwAwggEKAoIBAQDR8adUoBxw9L7E5rkMbvNmY2i5aJUecLSogXZ5arctuBeu4PQC
-2YzZU/0ff/kq8n/NhXATH11D7CXad7pYPbMaP1dqPkBdTAkRjkpwlkhGHQv/9teQ
-vMWesVr8rQ4ddk2DK/jmEEyFD3MwTgX8jsw7nhM+mBnhbvK/7DEEoAK6pfDuLl8f
-sVAHdc+3RClO0FQAYt1psvPB8Yvqx2NXy3GKRQn+wkr2pVysDkaEEqtgkE2KlD1Q
-fyqr5pnN1wDRtkO4syLLlQ9WECRaqU848FFWIH9GkJkx7nmRTEd0Ma/WS0O7BnXG
-xD3fRxNKtc7+PVxvdDb/roq2xiq96ww3f4IvAgMBAAGjUzBRMB0GA1UdDgQWBBSP
-+7lx7VdfCz4w2UGzgCsb1e69XzAfBgNVHSMEGDAWgBSP+7lx7VdfCz4w2UGzgCsb
-1e69XzAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDFBHvmndkO
-Up7scewQk8zACOcqyx33kUUT6NiAXDPglzITkVjkNU0r93aek6q5k4WmR2M2nm9n
-ptVRWMacZumxkT0wTqDEyH1cWBEjfNBBtI3JwRc2/6JCIEmqnlBc8/YOZ1xSh5/1
-T6x8dwNkR43wq1IoWilWIU8/trT97mly3X6yIzITzWEsGIiQmjzWvEJOg8tIjIsV
-d7JUDZ+MDC4Rhh/a7v6m7He3dchCsrsIVq/RiJyy8e8mbf0o0slB/SQrxbP0+lQ3
-TlE8Wk/FIpAptx0D5qUnq8E6CWxfUv+R6OK/trCQ+ILZgy2ReeM3yX5AZu7brKI3
-tHlEN2BVbdEo
+MIIDBTCCAe2gAwIBAgIUQExJNJG4q9PKcF6VZhx3N7QwAF0wDQYJKoZIhvcNAQEL
+BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzEyMDMyMTQ0MzdaFw0zMzEyMDEy
+MTQ0MzdaMBIxEDAOBgNVBAMMB1F0Uk8gQ0EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+DwAwggEKAoIBAQCXDWVfqxLFohypyfp/DS/tphmvt61c9WUAH7o34CmYXTjZlxkd
+tmhFHQznRW12oyt1Y78PQq0sbngL/uukb+3axYRIN4wQSLkAIVvdm6UbP7TnxzkC
+rGuEB0IEQuMKEGWYgKHwgwVcPovJjsqYPZVvyoixlBInz9FgdjbhlSPsFOSTVvZz
+K5Vzv6mO9Jff60P2ZoUcoiDrJLUznM6ITQ7+c38dZerOuBFV73N0CIsp4mO25//k
+qg0JXpmkAgt8TprOV3ngcuTCrdCe1SyLY850DO1cQj81HvSzlG6tkhfKKJx2NfKE
+MGhYl/DKc6er8q1ld2eAdNnMJ5Yr3UqQJRvjAgMBAAGjUzBRMB0GA1UdDgQWBBTv
+r1oi6bupMmgta/v+Dek0O9xCcDAfBgNVHSMEGDAWgBTvr1oi6bupMmgta/v+Dek0
+O9xCcDAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQBCWGZURlHB
+KfFfDSI4+4EkNax/LzYPpcI5w6HAQHG7RE5XanihJ4exE2Aqnwzu4UWZ/p7M2mPz
+pVN7NgGkqYN7TQBoRbyKwZO3KGw0x5ARm4SNW/T31I/u7zLAh66RXnMUJ46QaYH6
+hlJ2zusxp7saXgppx/ipJtJKd1IFIuKXMUS8uNEZ1Zly3ZMYt1udh6IYZqMcYbnG
+aEsvxMQq322mH4vGSonCXAnmuCESw/wR4PX+JQei3Z/0VYWhlxBKjyLrL0b9EQmK
+ps3wu1w4uuzWYiLNlDCQ+Ri6ej3APb9jhLg2KJ4SLB7UTvXN5/7CRf+pN182yo0n
+Xv1An8yXYqCH
 -----END CERTIFICATE-----
--- a/examples/remoteobjects/ssl/sslserver/cert/rootCA.srl
+++ b/examples/remoteobjects/ssl/sslserver/cert/rootCA.srl
@@ -1 +1 @@
-7390820BD47F5A3C7398B7C8DCA218B35C2C2E2B
+7390820BD47F5A3C7398B7C8DCA218B35C2C2E2F
--- a/examples/remoteobjects/ssl/sslserver/cert/server.crt
+++ b/examples/remoteobjects/ssl/sslserver/cert/server.crt
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDNTCCAh2gAwIBAgIUc5CCC9R/WjxzmLfI3KIYs1wsLiowDQYJKoZIhvcNAQEL
-BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzA1MzEwODExMTVaFw0yNTA5MDIw
-ODExMTVaMBQxEjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQAD
-ggEPADCCAQoCggEBAJIhl8T4lEioQYGoIFar95zkXi32PrRQ5cK36fP4mVgbJbmV
-ME1y6ZCG/29WMK9Ja3rQn2nkm4RiKVVd4/9WuVM7Eof7w1LRJ5kPxmfUFFc2WSvE
-VqazTikBUUYGW0otL0HEgyPBd39lexktJgk9AshGDOrcRZfNGn1SYlqh3g0V+TCz
-ZiN1drdIU19ZXWYBpzZcPibHxp0oTpDBIgiAD2TenzuLZkQPTuf+7BVchZ0Gy3nL
-VS5E1+M5uKjgaobOTdDLVjrHPgVTH/pa5ZCqELeRFVz6JyioC4HJy2HuZXeZNAQL
-zlqMAWNoUAMsvGKJVvg81ps0BmOrNKOaS7WzkZkCAwEAAaOBgDB+MB8GA1UdIwQY
-MBaAFI/7uXHtV18LPjDZQbOAKxvV7r1fMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgTw
+MIIDNTCCAh2gAwIBAgIUc5CCC9R/WjxzmLfI3KIYs1wsLi4wDQYJKoZIhvcNAQEL
+BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzEyMDMyMTQ0MzhaFw0zMzExMzAy
+MTQ0MzhaMBQxEjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQAD
+ggEPADCCAQoCggEBAKXeQUCDijqsGA3hTCwwD0fTa9afzuSrYJn3NzjeEOfNERnx
+iICjkMmHMXIw9s3NYAVDhOAQjXQLXdAWymelZq8tjOyrN3P+K6C6SrMl/bf88pTZ
+h49qdDSwjAcmhM0ZIqFEDfeUSMbV7JCiVa0ELPluKKAKcvUghHuPf9t/PhgHSdKU
+IXcifTX1RZzOBuHwuCOe+3TgxISaGZV/B4bdiOoIleiMHhL8CC07seLEp7gtYhh6
+l3pEpYStb66/6kSdliL9vTE10pR5XEsbu94aNBlEcLGxwHODRo2vfgkIAGQTuzGk
+aD9mlbmGb4o20cBoBmnSEPghR9KNq9Sgh+CemQcCAwEAAaOBgDB+MB8GA1UdIwQY
+MBaAFO+vWiLpu6kyaC1r+/4N6TQ73EJwMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgTw
 MBMGA1UdJQQMMAoGCCsGAQUFBwMBMA8GA1UdEQQIMAaHBH8AAAEwHQYDVR0OBBYE
-FOTMLTCG7GZ2LY1kn5tI7GTxn4R1MA0GCSqGSIb3DQEBCwUAA4IBAQC46Sr1neC2
-g9WikXFnptg6SvrX4xIowxV4f2y2iHSozcBz5lZYLRqLhsl0EuxO9UuCbqPx9nqi
-wHBk84S7METP+tmdvDBj7TtLEml7qwQIcR58YLD5Cu3oQLO1jcupq5DFAbKPBTBu
-ybJq+osofibogH6qdbYLyTjjFcTkg9A9VHvWEVhfaT355GmR4Yt+RKL9GiapDYF5
-ZiB8PRsX1HQzccmZsZrFVWb0klLq1tlDjSK1nx+xDBx7qbDLzUK49ie3k5MTVOfr
-E45yOZ6Sh8mHL+gCKcxFk83Wqy8oudChcGEiTEqKxDD268BvB00ECnhF3xAgtM+m
-LZ7CbX52p3bW
+FPkriomNYzq4cL13ZRpyrbYrHVJ2MA0GCSqGSIb3DQEBCwUAA4IBAQAGUGx/5Iiu
+9wDAASfT0xIp03sXkLjjE1Q+h2EYx/+4IoqeyziL5/xkEmqP7aROvgzwya+/z1Gw
+ycctqNPtKGHi6U4jcQQvl6OP9Ij9qae8yTtXufKzA/JJDh5YdOOgV2FqENKeANnd
+QlKoiXXfdtMQPAsK46UaBdL3pjprictUOgy1h7ItaqiPQsaeQ0e6CnjJ7LgOSiff
+oXHwTb9uRP6HuIuie+mG3ow7Q9BVzklHCRW4E1XeD1FiHBcKYDbEQ3TjFzsxd27T
+O3aivrczV79Km/InfaozssOT3cyYZc/BjKEwJZ7vECpmWZ0iYi0qzq5jF8qGLEDf
+1AqKTA1bN11Y
 -----END CERTIFICATE-----
--- a/examples/remoteobjects/ssl/sslserver/cert/server.key
+++ b/examples/remoteobjects/ssl/sslserver/cert/server.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCSIZfE+JRIqEGB
-qCBWq/ec5F4t9j60UOXCt+nz+JlYGyW5lTBNcumQhv9vVjCvSWt60J9p5JuEYilV
-XeP/VrlTOxKH+8NS0SeZD8Zn1BRXNlkrxFams04pAVFGBltKLS9BxIMjwXd/ZXsZ
-LSYJPQLIRgzq3EWXzRp9UmJaod4NFfkws2YjdXa3SFNfWV1mAac2XD4mx8adKE6Q
-wSIIgA9k3p87i2ZED07n/uwVXIWdBst5y1UuRNfjObio4GqGzk3Qy1Y6xz4FUx/6
-WuWQqhC3kRVc+icoqAuBycth7mV3mTQEC85ajAFjaFADLLxiiVb4PNabNAZjqzSj
-mku1s5GZAgMBAAECggEAC0J64QK9XD7Ys06a1UrRXL2s8xItATRVi9Qjevi+Cg+n
-b74K14Dh9dCtGG1V+X+pADDYg66JfCEbKbh/E6LhS0Q7OI6h3Xz/tdfFFCV/frvR
-kaGCgrxAqsCu/DwLpzLFmCX169ihpkeRvV4g9ufEUd2x65vW4ew8HQluyih8RVo8
-+spiAou5hPTT6rSX4A2ywthqyAdeq7hTEyIj6/Ai3gaSrL9htSJn4DxxYq5GiLjN
-WjhFUQWvz45fehLIRRA0yU+wUonGlI75KYasEd1O0cKttAePK0yEtaBuJbYCckYb
-PPD+iFU3KnBILNXVolBSlKOspAOtqDYDpKfSTdImgQKBgQC2lcFjO8lzzTy1Cw1Y
-9TMUR6yzjdQBvLpFVsB8kRQ/13bs/8FYiBhm9RbXE3SXCQ5Np6/FWlRAaGY8ieyg
-5+4B3+TqJvTfPrtS8tYBh8koai/Gc0Rq9jn95KrTe4pcMG+thodto84I/fwMzZHq
-Ns27/ejEjhdqviiGkWjJw3vwQQKBgQDM439pU4pLpzlDeyrSAPgAa4B6D48+BZpE
-Ap6Pb28Dsn3twl7Y8leodUeJIc+JVs1VRpzO8ROxtxGI2STebSHhBe1mEjhJpOPv
-OTwJTocdwd2xEwnxbz4OhNfnlN1ESTM1JZYl3XLANHqdxWb8fn7dYNjncdciOTqM
-LCa8obpLWQKBgCDJyd3BKc8i5yj34CC+f7q/mFLyQEVDmAvFpXF5ODKyxcePsi/s
-2Lz8fGqJMuHAnZHycA03yHN8jAeJGTZYEBTEGT63WjgQ8uDVfPrvYH9iekvf/URc
-DVP70XWC9PjTNerqgr18sg5JVg8Sh4aQ6uc8lKVfgvMHTQ3FZRT+hu5BAoGBALjK
-8y6nF1OWlnJDk2JnV0bhrrXodLRmzzVJ+4K46bK2+RdJOMyI/wDEDmojoZBoRrgE
-AQgwrSebSuhXmQY3/dd7xhtKkZPycVZhE8H9HqZ7JR1BY7VrNNhZfnWzcUHWl6X2
-qY6rRYUPkAvecD2JRZkbcWaF6ZhHb4PfimyVnXW5AoGAd8oiLmBV4AnBfv2YCDY4
-2Ec2+vcXrv4uS4xQq8KM7VLKKKACqOfmgMg//IcxF3EcXayigY711Gb60LVj5zX+
-bpl4ryS+pgH8qtMzGuWiNnUIBCQLp95vhz7WPDj2OmuRA6o+UiUg4bH146Kmq1+u
-mGYP0GQo5eyxWfXL0W5/fJY=
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCl3kFAg4o6rBgN
+4UwsMA9H02vWn87kq2CZ9zc43hDnzREZ8YiAo5DJhzFyMPbNzWAFQ4TgEI10C13Q
+FspnpWavLYzsqzdz/iugukqzJf23/PKU2YePanQ0sIwHJoTNGSKhRA33lEjG1eyQ
+olWtBCz5biigCnL1IIR7j3/bfz4YB0nSlCF3In019UWczgbh8Lgjnvt04MSEmhmV
+fweG3YjqCJXojB4S/AgtO7HixKe4LWIYepd6RKWErW+uv+pEnZYi/b0xNdKUeVxL
+G7veGjQZRHCxscBzg0aNr34JCABkE7sxpGg/ZpW5hm+KNtHAaAZp0hD4IUfSjavU
+oIfgnpkHAgMBAAECggEAGDdWNH/f5F1awlqvHyZscgftJG++1NuBJjQVz7YlfAo6
+9xwwhINyMJIN3IZHNNh0A+C4hhQJ6t7RSqpWe60GIbOXRw208g4yOsGSsAwZuk0c
+8jFoyBNP4EoZmjOmoOEMvBmuUQPztMqSbV1d/9LDvtjf59m5WzjCTt3ZBL1TiKNP
+NGX2NqXBwoKTLrHTWeMku63HOBHzDCTiRfqeCGaG/+i8/FHcz33ZBBuj9ZE6Pz5p
+6TnHgh0jU574vZmAEaWUkPUulO78onBOxjQgpGksbVJFdc5Ls6WRhJAkWJa8ozEM
+u1xzRyJu/O+r3YUXLzjBaB1KrJV1+0n06PUF1YfGrQKBgQDnED4C+LR8hAWuuQa/
+1DlW+1nmkYueMOa9DNIEoyx85yqwvZ/I6z/wEWT27j63ezWUJ5yumWNj7yjg6fze
+/FxtqdxMV8qEh7IgKrF0evgdUQ03naU9VMHzGFzvHgZE5D2a36mjbUG1ycdn1DXH
++l35WbnWE/c4qHnxQ3BunP7jSwKBgQC3xNDUZ8A3MErp7UNT4tGUKGdAPqYRd/gq
+Rw6EOpv782Qu6/sRIQvOo79ctCmelgBLjuNSRnUQPqFwesbcco0BAOTqJEmuMoga
+sy4UMt3t9T1egc0yA0G9ETQ6oL2STEhfsVgh6x52q8Gql7azqFDNXcF6FhvIkpAe
+QEgJdrOPtQKBgHdJ0VnjX/pwhiZ/61YZkEixluYL+dJ/WH/9IGe4ff01srpwCw/F
+M8sSj9dThBlhiHWmRaN2GPO8wpPq+Nt92ZHcAZmAMVNeYr1sjRSfb3xjHbm6r1YM
+bdy8lS31iDMEyjEsiv2OKEBbnZCi4+kL/nv+sIwu3pDnmZzOvjq83nIlAoGAFI50
+mb39V28UgjRjOqf9r6gRXLNlWkDkRr5qFkqEFlv0kfcdv53Ft1jD2DfjC/VVlR4n
+uZVeFPIclvt08UXzFfCmqrZic5T9epceiW3jO3Iaak70yCprGR4qjIBclFfqJKIu
+7KRjZPzzjgSokFI8r6xULzb35qjWbDvekwhvL50CgYBOQBuRSPD44sIkdpKq/Q8q
+51Vn/ggrHr2jNML+vKuyDfuPnlV3RiKtJ6dWt6omTxS8in8GJGpDgFG6eICuhVox
+vuOkm8bXjMZX37Hdog4iNG2xtUImrL2MTeVvGKWxFGR52/3YiqZ6jThA0A1RBCeL
+os0lSz56rZtnpMyG9Gi50w==
 -----END PRIVATE KEY-----
--- a/examples/remoteobjects/websockets/common/cert/client.crt
+++ b/examples/remoteobjects/websockets/common/cert/client.crt
@@ -1,17 +1,17 @@
 -----BEGIN CERTIFICATE-----
-MIICrTCCAZUCFHOQggvUf1o8c5i3yNyiGLNcLC4rMA0GCSqGSIb3DQEBCwUAMBIx
-EDAOBgNVBAMMB1F0Uk8gQ0EwHhcNMjMwNTMxMDgxMTE1WhcNMjUwOTAyMDgxMTE1
+MIICrTCCAZUCFHOQggvUf1o8c5i3yNyiGLNcLC4vMA0GCSqGSIb3DQEBCwUAMBIx
+EDAOBgNVBAMMB1F0Uk8gQ0EwHhcNMjMxMjAzMjE0NDM4WhcNMzMxMTMwMjE0NDM4
 WjAUMRIwEAYDVQQDDAkxMjcuMC4wLjEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAw
-ggEKAoIBAQDIP/hpIFwU9K1PolZWlgI3k8s8+ZqetECXlL6zf1ZLUWsDhPDeZDLT
-xXENiBnEnwomOXesB0G4lcD3r/oWZsEQK3mtukeZEsHRe/bllkqf3LuBcAOt62da
-it7On6cBSZNJLL6Hu1ctXi5L/mGoglDu9/2C//zJzj8KKjrDfbxMIuw2/guttYpM
-TzPNyrcA0OVaMAcw0niIbZlpJOn1q+sAuDXs1cAYYuNjkj6EUb7WnXJHDKUWu+q7
-WoBISfqH0p5bjEFz7VZOKJW25BXbJ6kpHMowBuyvNs+cTfWdRVNfVT+YJz0Vmyvb
-cUfFFEzmIIFJs9pH8nmHu6rpAgFfFOyhAgMBAAEwDQYJKoZIhvcNAQELBQADggEB
-AF0FzC0gRY0s6rAoJ+WV5WwCLahvUi4hR+Uug09rn1dgccDXMOcsFqGa9IKGaumq
-pWL/5aZzwt8izJ2LU3wYPE57bP21wRrVwvdIWX9CKab5DA8kL1BzQsoa+isS/beR
-dQjB2L9KZWRDkRVEJcyY/vfBkdJmwHTPBFx/EJjIwA93qfEvuif+bGL7xUVjIgxi
-MsSox3BJSdyNQ6MyoumCmUcD+iTCLQsavWpQuUv8nhXoTA2o2ozFSrDvWEuxwZyJ
-VRObyZLs1jJPQXAbKkv1DFFdlt3GlbS1TTjnlFMrZ77caDFX/CX01z37qqFTZw88
-S3FnpcsSk029XFie37tCYb8=
+ggEKAoIBAQC+b2yTU2pDQoS+fYKmOLVh/KkUgAU6ZV3g2ywmEdxJzF6knJqvwFui
+4MxfR9XACctEW+xxtklPy4aRFFWptYHi9v8V+Ac6SVEukGYP0A52Jn9/MqIZwu7e
+dwoWF1p5GVKkjz09FsnaOrWw8RRuD5jqcAheaURYRHiqveZxzxiBpmvgJpu2K1kf
+QS22fbnZGfRYdBcjhX+txOo3wt19qcgIofqh4M9N6gCwXt75VEEOuWwI8FCgX2y3
+n0p2nHM7iR4wPBYXJpV25K6FEM5AvQYkSjYpNwaggNTMv63xgtGEAPWC0XxA4Qfq
+Nxnj4qn4SU1lHr6nxVlDRox7yCUa58vbAgMBAAEwDQYJKoZIhvcNAQELBQADggEB
+AElzoUkYaV1uWPFZceYvEG2IOIl33yEMYqcRbdGRHimBA6hiupencxv+UG51aZ7Q
+un6sI01Nj4weeSQGWurl13qPoUJ5RvY295bDR2kS+qnb+9wG8H9/q/pq1yrlH5ZW
+IxWqJlPfUBKj/ShEplrDCn0OR38+tvNKAPi9PQVi9D8HD1SOtuhcLk4hU/Hqj9ek
+X/8kueT2bIc5u6rAoYktrQ3g0fqyfZUVezIqd0YlI9jUNRCfRd10Fyu+c8ODgCCS
+fXZ0HnBMWxfkF557O9q6MO2tWOJv+qq9qw9t5I/kduezkdGhMTeXv3NhxIHIYUTG
+CSLQbje++hKe5M7y5z6BoQ4=
 -----END CERTIFICATE-----
--- a/examples/remoteobjects/websockets/common/cert/client.key
+++ b/examples/remoteobjects/websockets/common/cert/client.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDIP/hpIFwU9K1P
-olZWlgI3k8s8+ZqetECXlL6zf1ZLUWsDhPDeZDLTxXENiBnEnwomOXesB0G4lcD3
-r/oWZsEQK3mtukeZEsHRe/bllkqf3LuBcAOt62dait7On6cBSZNJLL6Hu1ctXi5L
-/mGoglDu9/2C//zJzj8KKjrDfbxMIuw2/guttYpMTzPNyrcA0OVaMAcw0niIbZlp
-JOn1q+sAuDXs1cAYYuNjkj6EUb7WnXJHDKUWu+q7WoBISfqH0p5bjEFz7VZOKJW2
-5BXbJ6kpHMowBuyvNs+cTfWdRVNfVT+YJz0VmyvbcUfFFEzmIIFJs9pH8nmHu6rp
-AgFfFOyhAgMBAAECggEAAUU0nngb1W8y7UMmoJB9jZw0qRudQT37/eot5oYbhklL
-MD/G5mZE00aYn8EaBr8KibEKIw99uX3LtlaJs9wtg9x+vl0k6KHvlFRgvdac/RQh
-pWHQSClq465nWPC21sGAe3Ga+NH1rN5A1W8U8AR5nnZ4x3y3vhD+oeIgMs7Io8P/
-EiQIJoOWPFBtDR+7zSZ3W8zjXSHezI0fclhtXZIpwSiOnOSUUhzz73ObEU66tKa2
-LqA7WiQGWfV/wqMPOEeVBoD19HrgsinCvmpXtwsy69PpIk07Vdn7zbYkfHF49Uud
-/lfH3t4vAcnvJFlfPhgXuZzUBp7MSbk5y7EULnnwFQKBgQDxt6kOtwPIkpsHJz2i
-4Mlq053lnaH78xzwf3SH0HvcryiBps8Je5dBZ9xBwFHuwbn1cA7KzIs54Z8pBC3D
-fPDkT/X8a1pdpt9xZLvvmZ1GX2RGFR+gs1Ii5yNIrL5nsqYAoUaVGYJWBTXMskFO
-YHNVtmz418//orp7FsNyis2WbQKBgQDUFQ0Sy3IGwDlCohC9/bCm2Yu39d+2DPk5
-U60susxpQJcUhBlcv2rGhonLjwP2BilTEYqQ8IUN1uYnnYFw3lyJMZ5MxFQOVWrB
-/+Z9w10mRg2DgzPftrt6P+YNGTE8ygpai0JuAyUS0EPWB+LLIKPkLN/23gBbtZVD
-k4PBIhkehQKBgQDAG02bSk0RfaMEfmV4K9soseMN+Mgiq8dWRJXi0wl+DR/OWzq0
-uKsv6lGdWDJF5WXb8SFwgDwu1mheetm8QnJZoktIQDHjuJphTsQm2xmqMmpJYMRw
-PdSLtNPQEKqLU+qnGYY1Pb7eiumFNC6/EdzYRFI7O5ot+7Dl3uObB0H5yQKBgGMp
-sKIwFBHKATwBsRhUrStmnqiWQ1LJACLo2BkJApjYBYwnNACk09UdhQJzzWNUgDAC
-DKs2es/8wkKVp7dRkSzo0V6Nkcvd7nvlHi2BkmVZ7emnCDM7Kq+/ft5zAA2vXmX1
-pUrxF32TeZRyvX1Ctr7yqImyRbKb1b3yFHHScrJNAoGAc2eHAbmW3MjPwbHcdd8E
-ytL8wZhLmCBrAjUwv2qkAieh8meydLwNipl2gI95E1FmM4J5uFH8kB0WskcZ9vcC
-BK7He8JG9DoW8nHvupMxbHV3U30A65mfBn0C3/Oj+aHQtclXOo0odBSOzmghBn5F
-suxfBRpfio6OvYi192d7QCU=
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC+b2yTU2pDQoS+
+fYKmOLVh/KkUgAU6ZV3g2ywmEdxJzF6knJqvwFui4MxfR9XACctEW+xxtklPy4aR
+FFWptYHi9v8V+Ac6SVEukGYP0A52Jn9/MqIZwu7edwoWF1p5GVKkjz09FsnaOrWw
+8RRuD5jqcAheaURYRHiqveZxzxiBpmvgJpu2K1kfQS22fbnZGfRYdBcjhX+txOo3
+wt19qcgIofqh4M9N6gCwXt75VEEOuWwI8FCgX2y3n0p2nHM7iR4wPBYXJpV25K6F
+EM5AvQYkSjYpNwaggNTMv63xgtGEAPWC0XxA4QfqNxnj4qn4SU1lHr6nxVlDRox7
+yCUa58vbAgMBAAECggEAGoCGMzCSIr8gmyP7aSeOMo9Hdodkct6FhJPr3nZw2kJb
+T3xN2LSj7BRtzLrCCAI07BQA4UhVhonT9d/HFiGg7FWkZUnJs7ZdWmYVFhFF2fJ8
+ntaMkZo7xPD7Edz5Jc483pLimlxQBhNSksfSKIj6JnIl8AGlVcVdp6Fa7y67gdtK
+L6tJ+u3azUi9yry4VzNOLgQNZLaMQ//vuBjql8C/1Owq2AEdOlweTKZQ0NzKV3D4
+GL/nyakwX6dYqNz6//VXeaYK9hCOMFyO+w0GYTGrG8ryGSVoYMNRKib/DXCiFrHA
+9Vz9btZWp7M81NkuEymYdqcNJwciDwG7UYszqlZkxQKBgQDo99025fO8proFPVB3
+DbbfIb4mJuLi8OBweS7PTBg+8RugppAWLARrNN9EhmgNKu8SKCL02Tq3keqqx0uy
+L9zBIykpzA+LxPbvhXdNu98mTNNziwlLKO4kqhuC0j3FSgGhRLC7f4nZ6WWNEeGr
+6GChdnYO/0wl/WMcrZuwYVk1pwKBgQDRQxp8psrlGDH3t/57ezgLAHKQrQjBLwOO
+MQY/UrGkxtsG75HdLFtQNy0Yte79Y1byIVwoGgfduZQ5SNjSFcEfQhvuEpn09/sn
+PgYr9MLOxup56uPa3wAdmpaMpf50T1V+2PvrKGMNjgVV+MfG09ZTQahQDdluYbJA
+WPyNC5JmrQKBgQCgyfGz4iHHIotg8mCGysBcYD2Mt2oAl59QAVN9X1ecmWwoBM2N
+f/PMykEOWnWWIPu/3FPECemsqNj95W8pHD+QTWQrpNtV7CtCRSuCSxhIpt2J8XwI
+k8dJJt61+8UGVX2qgG45+lx6OcwNATpwG4IV7lytHSbBUZEGSgwaQBYCIQKBgE53
+OPFQoXex0NqwNU4b7/HAIzU4jLXTaSPiIhOQcGf24J4DKtCiF0NGzTQc1BZTL4Dv
+HV00FYsAUSIBvLtZyOZtpYdpli0S5yvZx/qndnyOfdFlUDb7s1Fma9klTPpT3R0J
+Q7zzkeC9RibseAyH1i+1yOTrrIIFRA94zoocsrKpAoGACywVD7XjcYG5GTvJ90L+
+UH5CnAD8vZZtZC2eYq37JxYgWs8yGooszlXfVXcYuQCZ8lPH8EEq+jD0QW/TJeAn
+q6LZVtmOsqTwhEaxxpJH/eh0SghfHDOUgaBxx7ufor0He4hciRnQqh43Dx1DMfRh
+23qWSBduVOdEUO+K9tHAMNg=
 -----END PRIVATE KEY-----
--- a/examples/remoteobjects/websockets/common/cert/rootCA.key
+++ b/examples/remoteobjects/websockets/common/cert/rootCA.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDR8adUoBxw9L7E
-5rkMbvNmY2i5aJUecLSogXZ5arctuBeu4PQC2YzZU/0ff/kq8n/NhXATH11D7CXa
-d7pYPbMaP1dqPkBdTAkRjkpwlkhGHQv/9teQvMWesVr8rQ4ddk2DK/jmEEyFD3Mw
-TgX8jsw7nhM+mBnhbvK/7DEEoAK6pfDuLl8fsVAHdc+3RClO0FQAYt1psvPB8Yvq
-x2NXy3GKRQn+wkr2pVysDkaEEqtgkE2KlD1Qfyqr5pnN1wDRtkO4syLLlQ9WECRa
-qU848FFWIH9GkJkx7nmRTEd0Ma/WS0O7BnXGxD3fRxNKtc7+PVxvdDb/roq2xiq9
-6ww3f4IvAgMBAAECggEAVsf/FPoCB6cDXDzK7RrHveEFnu5eBXEtHCI3qMS1504b
-bAxMkQkAMa2ICgdKJbiLn5ay1xS15qeslfu7qVIHwKfCKIaG8InHjdw1ZQiZ43GK
-TknonUmKnqhNJfkaxo7qR3hbsC24k5/pS2RVESqtvU8AQXV57TeBXFfzgyfpOOEz
-Nz4c27HfV5AZsDB4GOW1CS7ZdmvZuAfThhTJKh0y1Lqn/7TcVLEStDnFyUZig9gr
-I7WaxltruTgdmmhl5qX0Y1vgwS7APGGLYWTmRRBvyloyqfggCYeU3HDF3qH6RStK
-PsPbbHYW6Xj6LxZSJfVLdAVfFim0yAH1fTAmQXiOkQKBgQDdzLSA5zUBAIR3CTms
-sCn8a0ud8ETPUmTf1RmBMDeIuBTR3va/Q0YVYF8jY+ANrEwut0quIgOwZT4Kmo36
-Sx3F+uE7Gy0XQtDnZk7HeCOsF1fY7r81z7oAh61vFxG8ejSMqjUH8S3tTaCBvGi5
-z2ZjDX2lOsynkBwHu+jvYCbwRQKBgQDyUPWHkO41f6SoTXrEzf77y7Z5Wlim2Ltq
-gLAaEjLPuPsL+G/v7nCw6k6YmRli9f5jn1MGCzaRTxu1hkZlPhPvmfH/AGoSixgB
-CGKMCzOuFT3lQmpr0PZdJBpQ0aAo0R8X48J1X1WuyhZFRmafsrqcYeYPLLrs8aSp
-F51EiOJx4wKBgHZsxjPNP0+5pwCnvFOn+8L6xvUdRKAtrg6VCXkg9YXaQ8OK+NRG
-ZzUir6mFuiuSw1puI0G2h6G3bNjcjNIHKW579aZJbFnvIvtqdMWsL9/C1vTNBOsG
-bpm517Bo8N9hoI06k/YZoBwERp7+qiwLwO9CVMPUEVGmcFDw0/57qE7BAoGAXNwv
-q5sb3n6jAChpAuCada4UqJ652H64G9FjKyqMNFSESIsVgpISt4mjQxFwzB7PNShK
-bceASuFREhh6A7RteWW6emmTPLX3JfYrpdVpIa5cQO5Z0CWNNBZwy0X6JYo2azZK
-QCaftFMXg8a97zdUEjfmLSyMGd9kpbQ5/ZijhtcCgYBpmEOKN1SuGyEQ8gTz/02j
-Xz0XW63Dw8JX8s1Ck5u2uM96gsnHfwuC+iZTSu6FIS1e+3NOCEGQVy10rMzR/TLq
-h13ryz65qQtjaNl1gkD2Xf/ENxML57CU9nK5v+0zb+Qlk3CMKm0N06UmUTUDcE/T
-wIMUuL1LfdQ0b4bnS5JjjA==
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCXDWVfqxLFohyp
+yfp/DS/tphmvt61c9WUAH7o34CmYXTjZlxkdtmhFHQznRW12oyt1Y78PQq0sbngL
+/uukb+3axYRIN4wQSLkAIVvdm6UbP7TnxzkCrGuEB0IEQuMKEGWYgKHwgwVcPovJ
+jsqYPZVvyoixlBInz9FgdjbhlSPsFOSTVvZzK5Vzv6mO9Jff60P2ZoUcoiDrJLUz
+nM6ITQ7+c38dZerOuBFV73N0CIsp4mO25//kqg0JXpmkAgt8TprOV3ngcuTCrdCe
+1SyLY850DO1cQj81HvSzlG6tkhfKKJx2NfKEMGhYl/DKc6er8q1ld2eAdNnMJ5Yr
+3UqQJRvjAgMBAAECggEABCdyfe1hfcfrNowpycFeFmYMAiDGBsQaO/pOD71Sx4WW
+IP3JILUPLF9pJxm/XJTjhr3wgjCY50Y+d5ineFqPqwWQPBWIUjuobXrStkcn2kD9
+Q6aJ8ys0uxPMBVFQQcruUFcGnRZYA7prSRFr5utCpyllwE8h1LB1bKFeTwPf/LJn
+lFHLr34Vmr6whW3dSOn/QyjSRWXx6b88xc32yfniqI8JZrC5ul0J1ulGB0QRhGkB
+tOr2udYIZKjJkgYPQAL644NDEXL5B5iWUvzVzAyNTsLD/uSb4M/KwWxBlM1kvOGU
+GSNr/yKhwqHvV7zpAyjttEltRi44Nt9ULekEVUB+WQKBgQDHgC+vzZRisjhF46rn
+h5OkPMIWEWOAbJ0+fWFDGncDaTOc++QM1WNgJR24zODTKl92EwQHDVAMesPpO614
+DaoKarmvSqcSRDjKvcyij0GTnDe8qPYPn5aJd9vW4vSsqGNUlevClosCxiavSDrV
+Dyt48PejuX+UW3iqkl1QdTiKNwKBgQDB1LK1lcMrQJBY55lpsF2ovvsEDtwOPR93
+Icr3VtMpZTB0R442WzpvAhKaDLLaKUp5pzRGL4tN3pfpD2tz15+neKn4saE8zLoo
+1V0VDoBx9PCvfoP98/zbXZJiVFkdur+B6kz2ImUdnajL2lgnZjJSKGU3+QcEa8be
+vCEbuuI1tQKBgEtK57Rz5BKzrGVEHVYwZtpPS/jEK0dmtpnE/BY1QMF6zM3siw8E
+4fKBT4xVVmyby1dmqdh3f8SV+KZnrXpnvcscqGOtywwrzztFEpwvcNJi3O5I3kTb
+OBc9syyQj0MycfqUclwIMQKlAiiZHAsZJJEKcIvMEIhPVWUaxDj/ZI1VAoGBALuz
+JXwCkwUHeMEfRci7KXeHubysRxziw92xs8LhKYskmk+AG483uNZWw3y6X1g0NBKq
+gEyXLecybxkZ/YwoekrK5o4WNXWTK5LvSzxg1rGD6K+JbHg4Fdy4OFFQLRxFA77j
+DbQsio6rZhJCVquQo5o71Ajn9oupcuOuesFHsFchAoGAYedyPqf8UDgTbsPlA2uo
+kzPanG7Vb5JtHbE14XsAaQP27EX6Ww96oCjEMEhq3KqHj+CQGga0uj6dhgWw+8fl
+s8sLz456U6XkRNjjDJ5luklxvNY2ejETcnZ16t0dp8TtJ8xiFvYO3XyIioCMdtY9
+3TVra0Izy2Wgc1dg4Pz9VKA=
 -----END PRIVATE KEY-----
--- a/examples/remoteobjects/websockets/common/cert/rootCA.pem
+++ b/examples/remoteobjects/websockets/common/cert/rootCA.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDBTCCAe2gAwIBAgIURtkt3RNfea7kp8Dl9UBvr0mXk0kwDQYJKoZIhvcNAQEL
-BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzA1MzEwODExMTRaFw0yNTA5MTMw
-ODExMTRaMBIxEDAOBgNVBAMMB1F0Uk8gQ0EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
-DwAwggEKAoIBAQDR8adUoBxw9L7E5rkMbvNmY2i5aJUecLSogXZ5arctuBeu4PQC
-2YzZU/0ff/kq8n/NhXATH11D7CXad7pYPbMaP1dqPkBdTAkRjkpwlkhGHQv/9teQ
-vMWesVr8rQ4ddk2DK/jmEEyFD3MwTgX8jsw7nhM+mBnhbvK/7DEEoAK6pfDuLl8f
-sVAHdc+3RClO0FQAYt1psvPB8Yvqx2NXy3GKRQn+wkr2pVysDkaEEqtgkE2KlD1Q
-fyqr5pnN1wDRtkO4syLLlQ9WECRaqU848FFWIH9GkJkx7nmRTEd0Ma/WS0O7BnXG
-xD3fRxNKtc7+PVxvdDb/roq2xiq96ww3f4IvAgMBAAGjUzBRMB0GA1UdDgQWBBSP
-+7lx7VdfCz4w2UGzgCsb1e69XzAfBgNVHSMEGDAWgBSP+7lx7VdfCz4w2UGzgCsb
-1e69XzAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDFBHvmndkO
-Up7scewQk8zACOcqyx33kUUT6NiAXDPglzITkVjkNU0r93aek6q5k4WmR2M2nm9n
-ptVRWMacZumxkT0wTqDEyH1cWBEjfNBBtI3JwRc2/6JCIEmqnlBc8/YOZ1xSh5/1
-T6x8dwNkR43wq1IoWilWIU8/trT97mly3X6yIzITzWEsGIiQmjzWvEJOg8tIjIsV
-d7JUDZ+MDC4Rhh/a7v6m7He3dchCsrsIVq/RiJyy8e8mbf0o0slB/SQrxbP0+lQ3
-TlE8Wk/FIpAptx0D5qUnq8E6CWxfUv+R6OK/trCQ+ILZgy2ReeM3yX5AZu7brKI3
-tHlEN2BVbdEo
+MIIDBTCCAe2gAwIBAgIUQExJNJG4q9PKcF6VZhx3N7QwAF0wDQYJKoZIhvcNAQEL
+BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzEyMDMyMTQ0MzdaFw0zMzEyMDEy
+MTQ0MzdaMBIxEDAOBgNVBAMMB1F0Uk8gQ0EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+DwAwggEKAoIBAQCXDWVfqxLFohypyfp/DS/tphmvt61c9WUAH7o34CmYXTjZlxkd
+tmhFHQznRW12oyt1Y78PQq0sbngL/uukb+3axYRIN4wQSLkAIVvdm6UbP7TnxzkC
+rGuEB0IEQuMKEGWYgKHwgwVcPovJjsqYPZVvyoixlBInz9FgdjbhlSPsFOSTVvZz
+K5Vzv6mO9Jff60P2ZoUcoiDrJLUznM6ITQ7+c38dZerOuBFV73N0CIsp4mO25//k
+qg0JXpmkAgt8TprOV3ngcuTCrdCe1SyLY850DO1cQj81HvSzlG6tkhfKKJx2NfKE
+MGhYl/DKc6er8q1ld2eAdNnMJ5Yr3UqQJRvjAgMBAAGjUzBRMB0GA1UdDgQWBBTv
+r1oi6bupMmgta/v+Dek0O9xCcDAfBgNVHSMEGDAWgBTvr1oi6bupMmgta/v+Dek0
+O9xCcDAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQBCWGZURlHB
+KfFfDSI4+4EkNax/LzYPpcI5w6HAQHG7RE5XanihJ4exE2Aqnwzu4UWZ/p7M2mPz
+pVN7NgGkqYN7TQBoRbyKwZO3KGw0x5ARm4SNW/T31I/u7zLAh66RXnMUJ46QaYH6
+hlJ2zusxp7saXgppx/ipJtJKd1IFIuKXMUS8uNEZ1Zly3ZMYt1udh6IYZqMcYbnG
+aEsvxMQq322mH4vGSonCXAnmuCESw/wR4PX+JQei3Z/0VYWhlxBKjyLrL0b9EQmK
+ps3wu1w4uuzWYiLNlDCQ+Ri6ej3APb9jhLg2KJ4SLB7UTvXN5/7CRf+pN182yo0n
+Xv1An8yXYqCH
 -----END CERTIFICATE-----
--- a/examples/remoteobjects/websockets/common/cert/rootCA.srl
+++ b/examples/remoteobjects/websockets/common/cert/rootCA.srl
@@ -1 +1 @@
-7390820BD47F5A3C7398B7C8DCA218B35C2C2E2B
+7390820BD47F5A3C7398B7C8DCA218B35C2C2E2F
--- a/examples/remoteobjects/websockets/common/cert/server.crt
+++ b/examples/remoteobjects/websockets/common/cert/server.crt
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDNTCCAh2gAwIBAgIUc5CCC9R/WjxzmLfI3KIYs1wsLiowDQYJKoZIhvcNAQEL
-BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzA1MzEwODExMTVaFw0yNTA5MDIw
-ODExMTVaMBQxEjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQAD
-ggEPADCCAQoCggEBAJIhl8T4lEioQYGoIFar95zkXi32PrRQ5cK36fP4mVgbJbmV
-ME1y6ZCG/29WMK9Ja3rQn2nkm4RiKVVd4/9WuVM7Eof7w1LRJ5kPxmfUFFc2WSvE
-VqazTikBUUYGW0otL0HEgyPBd39lexktJgk9AshGDOrcRZfNGn1SYlqh3g0V+TCz
-ZiN1drdIU19ZXWYBpzZcPibHxp0oTpDBIgiAD2TenzuLZkQPTuf+7BVchZ0Gy3nL
-VS5E1+M5uKjgaobOTdDLVjrHPgVTH/pa5ZCqELeRFVz6JyioC4HJy2HuZXeZNAQL
-zlqMAWNoUAMsvGKJVvg81ps0BmOrNKOaS7WzkZkCAwEAAaOBgDB+MB8GA1UdIwQY
-MBaAFI/7uXHtV18LPjDZQbOAKxvV7r1fMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgTw
+MIIDNTCCAh2gAwIBAgIUc5CCC9R/WjxzmLfI3KIYs1wsLi4wDQYJKoZIhvcNAQEL
+BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzEyMDMyMTQ0MzhaFw0zMzExMzAy
+MTQ0MzhaMBQxEjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQAD
+ggEPADCCAQoCggEBAKXeQUCDijqsGA3hTCwwD0fTa9afzuSrYJn3NzjeEOfNERnx
+iICjkMmHMXIw9s3NYAVDhOAQjXQLXdAWymelZq8tjOyrN3P+K6C6SrMl/bf88pTZ
+h49qdDSwjAcmhM0ZIqFEDfeUSMbV7JCiVa0ELPluKKAKcvUghHuPf9t/PhgHSdKU
+IXcifTX1RZzOBuHwuCOe+3TgxISaGZV/B4bdiOoIleiMHhL8CC07seLEp7gtYhh6
+l3pEpYStb66/6kSdliL9vTE10pR5XEsbu94aNBlEcLGxwHODRo2vfgkIAGQTuzGk
+aD9mlbmGb4o20cBoBmnSEPghR9KNq9Sgh+CemQcCAwEAAaOBgDB+MB8GA1UdIwQY
+MBaAFO+vWiLpu6kyaC1r+/4N6TQ73EJwMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgTw
 MBMGA1UdJQQMMAoGCCsGAQUFBwMBMA8GA1UdEQQIMAaHBH8AAAEwHQYDVR0OBBYE
-FOTMLTCG7GZ2LY1kn5tI7GTxn4R1MA0GCSqGSIb3DQEBCwUAA4IBAQC46Sr1neC2
-g9WikXFnptg6SvrX4xIowxV4f2y2iHSozcBz5lZYLRqLhsl0EuxO9UuCbqPx9nqi
-wHBk84S7METP+tmdvDBj7TtLEml7qwQIcR58YLD5Cu3oQLO1jcupq5DFAbKPBTBu
-ybJq+osofibogH6qdbYLyTjjFcTkg9A9VHvWEVhfaT355GmR4Yt+RKL9GiapDYF5
-ZiB8PRsX1HQzccmZsZrFVWb0klLq1tlDjSK1nx+xDBx7qbDLzUK49ie3k5MTVOfr
-E45yOZ6Sh8mHL+gCKcxFk83Wqy8oudChcGEiTEqKxDD268BvB00ECnhF3xAgtM+m
-LZ7CbX52p3bW
+FPkriomNYzq4cL13ZRpyrbYrHVJ2MA0GCSqGSIb3DQEBCwUAA4IBAQAGUGx/5Iiu
+9wDAASfT0xIp03sXkLjjE1Q+h2EYx/+4IoqeyziL5/xkEmqP7aROvgzwya+/z1Gw
+ycctqNPtKGHi6U4jcQQvl6OP9Ij9qae8yTtXufKzA/JJDh5YdOOgV2FqENKeANnd
+QlKoiXXfdtMQPAsK46UaBdL3pjprictUOgy1h7ItaqiPQsaeQ0e6CnjJ7LgOSiff
+oXHwTb9uRP6HuIuie+mG3ow7Q9BVzklHCRW4E1XeD1FiHBcKYDbEQ3TjFzsxd27T
+O3aivrczV79Km/InfaozssOT3cyYZc/BjKEwJZ7vECpmWZ0iYi0qzq5jF8qGLEDf
+1AqKTA1bN11Y
 -----END CERTIFICATE-----
--- a/examples/remoteobjects/websockets/common/cert/server.key
+++ b/examples/remoteobjects/websockets/common/cert/server.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCSIZfE+JRIqEGB
-qCBWq/ec5F4t9j60UOXCt+nz+JlYGyW5lTBNcumQhv9vVjCvSWt60J9p5JuEYilV
-XeP/VrlTOxKH+8NS0SeZD8Zn1BRXNlkrxFams04pAVFGBltKLS9BxIMjwXd/ZXsZ
-LSYJPQLIRgzq3EWXzRp9UmJaod4NFfkws2YjdXa3SFNfWV1mAac2XD4mx8adKE6Q
-wSIIgA9k3p87i2ZED07n/uwVXIWdBst5y1UuRNfjObio4GqGzk3Qy1Y6xz4FUx/6
-WuWQqhC3kRVc+icoqAuBycth7mV3mTQEC85ajAFjaFADLLxiiVb4PNabNAZjqzSj
-mku1s5GZAgMBAAECggEAC0J64QK9XD7Ys06a1UrRXL2s8xItATRVi9Qjevi+Cg+n
-b74K14Dh9dCtGG1V+X+pADDYg66JfCEbKbh/E6LhS0Q7OI6h3Xz/tdfFFCV/frvR
-kaGCgrxAqsCu/DwLpzLFmCX169ihpkeRvV4g9ufEUd2x65vW4ew8HQluyih8RVo8
-+spiAou5hPTT6rSX4A2ywthqyAdeq7hTEyIj6/Ai3gaSrL9htSJn4DxxYq5GiLjN
-WjhFUQWvz45fehLIRRA0yU+wUonGlI75KYasEd1O0cKttAePK0yEtaBuJbYCckYb
-PPD+iFU3KnBILNXVolBSlKOspAOtqDYDpKfSTdImgQKBgQC2lcFjO8lzzTy1Cw1Y
-9TMUR6yzjdQBvLpFVsB8kRQ/13bs/8FYiBhm9RbXE3SXCQ5Np6/FWlRAaGY8ieyg
-5+4B3+TqJvTfPrtS8tYBh8koai/Gc0Rq9jn95KrTe4pcMG+thodto84I/fwMzZHq
-Ns27/ejEjhdqviiGkWjJw3vwQQKBgQDM439pU4pLpzlDeyrSAPgAa4B6D48+BZpE
-Ap6Pb28Dsn3twl7Y8leodUeJIc+JVs1VRpzO8ROxtxGI2STebSHhBe1mEjhJpOPv
-OTwJTocdwd2xEwnxbz4OhNfnlN1ESTM1JZYl3XLANHqdxWb8fn7dYNjncdciOTqM
-LCa8obpLWQKBgCDJyd3BKc8i5yj34CC+f7q/mFLyQEVDmAvFpXF5ODKyxcePsi/s
-2Lz8fGqJMuHAnZHycA03yHN8jAeJGTZYEBTEGT63WjgQ8uDVfPrvYH9iekvf/URc
-DVP70XWC9PjTNerqgr18sg5JVg8Sh4aQ6uc8lKVfgvMHTQ3FZRT+hu5BAoGBALjK
-8y6nF1OWlnJDk2JnV0bhrrXodLRmzzVJ+4K46bK2+RdJOMyI/wDEDmojoZBoRrgE
-AQgwrSebSuhXmQY3/dd7xhtKkZPycVZhE8H9HqZ7JR1BY7VrNNhZfnWzcUHWl6X2
-qY6rRYUPkAvecD2JRZkbcWaF6ZhHb4PfimyVnXW5AoGAd8oiLmBV4AnBfv2YCDY4
-2Ec2+vcXrv4uS4xQq8KM7VLKKKACqOfmgMg//IcxF3EcXayigY711Gb60LVj5zX+
-bpl4ryS+pgH8qtMzGuWiNnUIBCQLp95vhz7WPDj2OmuRA6o+UiUg4bH146Kmq1+u
-mGYP0GQo5eyxWfXL0W5/fJY=
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCl3kFAg4o6rBgN
+4UwsMA9H02vWn87kq2CZ9zc43hDnzREZ8YiAo5DJhzFyMPbNzWAFQ4TgEI10C13Q
+FspnpWavLYzsqzdz/iugukqzJf23/PKU2YePanQ0sIwHJoTNGSKhRA33lEjG1eyQ
+olWtBCz5biigCnL1IIR7j3/bfz4YB0nSlCF3In019UWczgbh8Lgjnvt04MSEmhmV
+fweG3YjqCJXojB4S/AgtO7HixKe4LWIYepd6RKWErW+uv+pEnZYi/b0xNdKUeVxL
+G7veGjQZRHCxscBzg0aNr34JCABkE7sxpGg/ZpW5hm+KNtHAaAZp0hD4IUfSjavU
+oIfgnpkHAgMBAAECggEAGDdWNH/f5F1awlqvHyZscgftJG++1NuBJjQVz7YlfAo6
+9xwwhINyMJIN3IZHNNh0A+C4hhQJ6t7RSqpWe60GIbOXRw208g4yOsGSsAwZuk0c
+8jFoyBNP4EoZmjOmoOEMvBmuUQPztMqSbV1d/9LDvtjf59m5WzjCTt3ZBL1TiKNP
+NGX2NqXBwoKTLrHTWeMku63HOBHzDCTiRfqeCGaG/+i8/FHcz33ZBBuj9ZE6Pz5p
+6TnHgh0jU574vZmAEaWUkPUulO78onBOxjQgpGksbVJFdc5Ls6WRhJAkWJa8ozEM
+u1xzRyJu/O+r3YUXLzjBaB1KrJV1+0n06PUF1YfGrQKBgQDnED4C+LR8hAWuuQa/
+1DlW+1nmkYueMOa9DNIEoyx85yqwvZ/I6z/wEWT27j63ezWUJ5yumWNj7yjg6fze
+/FxtqdxMV8qEh7IgKrF0evgdUQ03naU9VMHzGFzvHgZE5D2a36mjbUG1ycdn1DXH
++l35WbnWE/c4qHnxQ3BunP7jSwKBgQC3xNDUZ8A3MErp7UNT4tGUKGdAPqYRd/gq
+Rw6EOpv782Qu6/sRIQvOo79ctCmelgBLjuNSRnUQPqFwesbcco0BAOTqJEmuMoga
+sy4UMt3t9T1egc0yA0G9ETQ6oL2STEhfsVgh6x52q8Gql7azqFDNXcF6FhvIkpAe
+QEgJdrOPtQKBgHdJ0VnjX/pwhiZ/61YZkEixluYL+dJ/WH/9IGe4ff01srpwCw/F
+M8sSj9dThBlhiHWmRaN2GPO8wpPq+Nt92ZHcAZmAMVNeYr1sjRSfb3xjHbm6r1YM
+bdy8lS31iDMEyjEsiv2OKEBbnZCi4+kL/nv+sIwu3pDnmZzOvjq83nIlAoGAFI50
+mb39V28UgjRjOqf9r6gRXLNlWkDkRr5qFkqEFlv0kfcdv53Ft1jD2DfjC/VVlR4n
+uZVeFPIclvt08UXzFfCmqrZic5T9epceiW3jO3Iaak70yCprGR4qjIBclFfqJKIu
+7KRjZPzzjgSokFI8r6xULzb35qjWbDvekwhvL50CgYBOQBuRSPD44sIkdpKq/Q8q
+51Vn/ggrHr2jNML+vKuyDfuPnlV3RiKtJ6dWt6omTxS8in8GJGpDgFG6eICuhVox
+vuOkm8bXjMZX37Hdog4iNG2xtUImrL2MTeVvGKWxFGR52/3YiqZ6jThA0A1RBCeL
+os0lSz56rZtnpMyG9Gi50w==
 -----END PRIVATE KEY-----
--- a/tests/auto/external_IODevice/cert/client.crt
+++ b/tests/auto/external_IODevice/cert/client.crt
@@ -1,17 +1,17 @@
 -----BEGIN CERTIFICATE-----
-MIICrTCCAZUCFHOQggvUf1o8c5i3yNyiGLNcLC4rMA0GCSqGSIb3DQEBCwUAMBIx
-EDAOBgNVBAMMB1F0Uk8gQ0EwHhcNMjMwNTMxMDgxMTE1WhcNMjUwOTAyMDgxMTE1
+MIICrTCCAZUCFHOQggvUf1o8c5i3yNyiGLNcLC4vMA0GCSqGSIb3DQEBCwUAMBIx
+EDAOBgNVBAMMB1F0Uk8gQ0EwHhcNMjMxMjAzMjE0NDM4WhcNMzMxMTMwMjE0NDM4
 WjAUMRIwEAYDVQQDDAkxMjcuMC4wLjEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAw
-ggEKAoIBAQDIP/hpIFwU9K1PolZWlgI3k8s8+ZqetECXlL6zf1ZLUWsDhPDeZDLT
-xXENiBnEnwomOXesB0G4lcD3r/oWZsEQK3mtukeZEsHRe/bllkqf3LuBcAOt62da
-it7On6cBSZNJLL6Hu1ctXi5L/mGoglDu9/2C//zJzj8KKjrDfbxMIuw2/guttYpM
-TzPNyrcA0OVaMAcw0niIbZlpJOn1q+sAuDXs1cAYYuNjkj6EUb7WnXJHDKUWu+q7
-WoBISfqH0p5bjEFz7VZOKJW25BXbJ6kpHMowBuyvNs+cTfWdRVNfVT+YJz0Vmyvb
-cUfFFEzmIIFJs9pH8nmHu6rpAgFfFOyhAgMBAAEwDQYJKoZIhvcNAQELBQADggEB
-AF0FzC0gRY0s6rAoJ+WV5WwCLahvUi4hR+Uug09rn1dgccDXMOcsFqGa9IKGaumq
-pWL/5aZzwt8izJ2LU3wYPE57bP21wRrVwvdIWX9CKab5DA8kL1BzQsoa+isS/beR
-dQjB2L9KZWRDkRVEJcyY/vfBkdJmwHTPBFx/EJjIwA93qfEvuif+bGL7xUVjIgxi
-MsSox3BJSdyNQ6MyoumCmUcD+iTCLQsavWpQuUv8nhXoTA2o2ozFSrDvWEuxwZyJ
-VRObyZLs1jJPQXAbKkv1DFFdlt3GlbS1TTjnlFMrZ77caDFX/CX01z37qqFTZw88
-S3FnpcsSk029XFie37tCYb8=
+ggEKAoIBAQC+b2yTU2pDQoS+fYKmOLVh/KkUgAU6ZV3g2ywmEdxJzF6knJqvwFui
+4MxfR9XACctEW+xxtklPy4aRFFWptYHi9v8V+Ac6SVEukGYP0A52Jn9/MqIZwu7e
+dwoWF1p5GVKkjz09FsnaOrWw8RRuD5jqcAheaURYRHiqveZxzxiBpmvgJpu2K1kf
+QS22fbnZGfRYdBcjhX+txOo3wt19qcgIofqh4M9N6gCwXt75VEEOuWwI8FCgX2y3
+n0p2nHM7iR4wPBYXJpV25K6FEM5AvQYkSjYpNwaggNTMv63xgtGEAPWC0XxA4Qfq
+Nxnj4qn4SU1lHr6nxVlDRox7yCUa58vbAgMBAAEwDQYJKoZIhvcNAQELBQADggEB
+AElzoUkYaV1uWPFZceYvEG2IOIl33yEMYqcRbdGRHimBA6hiupencxv+UG51aZ7Q
+un6sI01Nj4weeSQGWurl13qPoUJ5RvY295bDR2kS+qnb+9wG8H9/q/pq1yrlH5ZW
+IxWqJlPfUBKj/ShEplrDCn0OR38+tvNKAPi9PQVi9D8HD1SOtuhcLk4hU/Hqj9ek
+X/8kueT2bIc5u6rAoYktrQ3g0fqyfZUVezIqd0YlI9jUNRCfRd10Fyu+c8ODgCCS
+fXZ0HnBMWxfkF557O9q6MO2tWOJv+qq9qw9t5I/kduezkdGhMTeXv3NhxIHIYUTG
+CSLQbje++hKe5M7y5z6BoQ4=
 -----END CERTIFICATE-----
--- a/tests/auto/external_IODevice/cert/client.key
+++ b/tests/auto/external_IODevice/cert/client.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDIP/hpIFwU9K1P
-olZWlgI3k8s8+ZqetECXlL6zf1ZLUWsDhPDeZDLTxXENiBnEnwomOXesB0G4lcD3
-r/oWZsEQK3mtukeZEsHRe/bllkqf3LuBcAOt62dait7On6cBSZNJLL6Hu1ctXi5L
-/mGoglDu9/2C//zJzj8KKjrDfbxMIuw2/guttYpMTzPNyrcA0OVaMAcw0niIbZlp
-JOn1q+sAuDXs1cAYYuNjkj6EUb7WnXJHDKUWu+q7WoBISfqH0p5bjEFz7VZOKJW2
-5BXbJ6kpHMowBuyvNs+cTfWdRVNfVT+YJz0VmyvbcUfFFEzmIIFJs9pH8nmHu6rp
-AgFfFOyhAgMBAAECggEAAUU0nngb1W8y7UMmoJB9jZw0qRudQT37/eot5oYbhklL
-MD/G5mZE00aYn8EaBr8KibEKIw99uX3LtlaJs9wtg9x+vl0k6KHvlFRgvdac/RQh
-pWHQSClq465nWPC21sGAe3Ga+NH1rN5A1W8U8AR5nnZ4x3y3vhD+oeIgMs7Io8P/
-EiQIJoOWPFBtDR+7zSZ3W8zjXSHezI0fclhtXZIpwSiOnOSUUhzz73ObEU66tKa2
-LqA7WiQGWfV/wqMPOEeVBoD19HrgsinCvmpXtwsy69PpIk07Vdn7zbYkfHF49Uud
-/lfH3t4vAcnvJFlfPhgXuZzUBp7MSbk5y7EULnnwFQKBgQDxt6kOtwPIkpsHJz2i
-4Mlq053lnaH78xzwf3SH0HvcryiBps8Je5dBZ9xBwFHuwbn1cA7KzIs54Z8pBC3D
-fPDkT/X8a1pdpt9xZLvvmZ1GX2RGFR+gs1Ii5yNIrL5nsqYAoUaVGYJWBTXMskFO
-YHNVtmz418//orp7FsNyis2WbQKBgQDUFQ0Sy3IGwDlCohC9/bCm2Yu39d+2DPk5
-U60susxpQJcUhBlcv2rGhonLjwP2BilTEYqQ8IUN1uYnnYFw3lyJMZ5MxFQOVWrB
-/+Z9w10mRg2DgzPftrt6P+YNGTE8ygpai0JuAyUS0EPWB+LLIKPkLN/23gBbtZVD
-k4PBIhkehQKBgQDAG02bSk0RfaMEfmV4K9soseMN+Mgiq8dWRJXi0wl+DR/OWzq0
-uKsv6lGdWDJF5WXb8SFwgDwu1mheetm8QnJZoktIQDHjuJphTsQm2xmqMmpJYMRw
-PdSLtNPQEKqLU+qnGYY1Pb7eiumFNC6/EdzYRFI7O5ot+7Dl3uObB0H5yQKBgGMp
-sKIwFBHKATwBsRhUrStmnqiWQ1LJACLo2BkJApjYBYwnNACk09UdhQJzzWNUgDAC
-DKs2es/8wkKVp7dRkSzo0V6Nkcvd7nvlHi2BkmVZ7emnCDM7Kq+/ft5zAA2vXmX1
-pUrxF32TeZRyvX1Ctr7yqImyRbKb1b3yFHHScrJNAoGAc2eHAbmW3MjPwbHcdd8E
-ytL8wZhLmCBrAjUwv2qkAieh8meydLwNipl2gI95E1FmM4J5uFH8kB0WskcZ9vcC
-BK7He8JG9DoW8nHvupMxbHV3U30A65mfBn0C3/Oj+aHQtclXOo0odBSOzmghBn5F
-suxfBRpfio6OvYi192d7QCU=
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC+b2yTU2pDQoS+
+fYKmOLVh/KkUgAU6ZV3g2ywmEdxJzF6knJqvwFui4MxfR9XACctEW+xxtklPy4aR
+FFWptYHi9v8V+Ac6SVEukGYP0A52Jn9/MqIZwu7edwoWF1p5GVKkjz09FsnaOrWw
+8RRuD5jqcAheaURYRHiqveZxzxiBpmvgJpu2K1kfQS22fbnZGfRYdBcjhX+txOo3
+wt19qcgIofqh4M9N6gCwXt75VEEOuWwI8FCgX2y3n0p2nHM7iR4wPBYXJpV25K6F
+EM5AvQYkSjYpNwaggNTMv63xgtGEAPWC0XxA4QfqNxnj4qn4SU1lHr6nxVlDRox7
+yCUa58vbAgMBAAECggEAGoCGMzCSIr8gmyP7aSeOMo9Hdodkct6FhJPr3nZw2kJb
+T3xN2LSj7BRtzLrCCAI07BQA4UhVhonT9d/HFiGg7FWkZUnJs7ZdWmYVFhFF2fJ8
+ntaMkZo7xPD7Edz5Jc483pLimlxQBhNSksfSKIj6JnIl8AGlVcVdp6Fa7y67gdtK
+L6tJ+u3azUi9yry4VzNOLgQNZLaMQ//vuBjql8C/1Owq2AEdOlweTKZQ0NzKV3D4
+GL/nyakwX6dYqNz6//VXeaYK9hCOMFyO+w0GYTGrG8ryGSVoYMNRKib/DXCiFrHA
+9Vz9btZWp7M81NkuEymYdqcNJwciDwG7UYszqlZkxQKBgQDo99025fO8proFPVB3
+DbbfIb4mJuLi8OBweS7PTBg+8RugppAWLARrNN9EhmgNKu8SKCL02Tq3keqqx0uy
+L9zBIykpzA+LxPbvhXdNu98mTNNziwlLKO4kqhuC0j3FSgGhRLC7f4nZ6WWNEeGr
+6GChdnYO/0wl/WMcrZuwYVk1pwKBgQDRQxp8psrlGDH3t/57ezgLAHKQrQjBLwOO
+MQY/UrGkxtsG75HdLFtQNy0Yte79Y1byIVwoGgfduZQ5SNjSFcEfQhvuEpn09/sn
+PgYr9MLOxup56uPa3wAdmpaMpf50T1V+2PvrKGMNjgVV+MfG09ZTQahQDdluYbJA
+WPyNC5JmrQKBgQCgyfGz4iHHIotg8mCGysBcYD2Mt2oAl59QAVN9X1ecmWwoBM2N
+f/PMykEOWnWWIPu/3FPECemsqNj95W8pHD+QTWQrpNtV7CtCRSuCSxhIpt2J8XwI
+k8dJJt61+8UGVX2qgG45+lx6OcwNATpwG4IV7lytHSbBUZEGSgwaQBYCIQKBgE53
+OPFQoXex0NqwNU4b7/HAIzU4jLXTaSPiIhOQcGf24J4DKtCiF0NGzTQc1BZTL4Dv
+HV00FYsAUSIBvLtZyOZtpYdpli0S5yvZx/qndnyOfdFlUDb7s1Fma9klTPpT3R0J
+Q7zzkeC9RibseAyH1i+1yOTrrIIFRA94zoocsrKpAoGACywVD7XjcYG5GTvJ90L+
+UH5CnAD8vZZtZC2eYq37JxYgWs8yGooszlXfVXcYuQCZ8lPH8EEq+jD0QW/TJeAn
+q6LZVtmOsqTwhEaxxpJH/eh0SghfHDOUgaBxx7ufor0He4hciRnQqh43Dx1DMfRh
+23qWSBduVOdEUO+K9tHAMNg=
 -----END PRIVATE KEY-----
--- a/tests/auto/external_IODevice/cert/rootCA.key
+++ b/tests/auto/external_IODevice/cert/rootCA.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDR8adUoBxw9L7E
-5rkMbvNmY2i5aJUecLSogXZ5arctuBeu4PQC2YzZU/0ff/kq8n/NhXATH11D7CXa
-d7pYPbMaP1dqPkBdTAkRjkpwlkhGHQv/9teQvMWesVr8rQ4ddk2DK/jmEEyFD3Mw
-TgX8jsw7nhM+mBnhbvK/7DEEoAK6pfDuLl8fsVAHdc+3RClO0FQAYt1psvPB8Yvq
-x2NXy3GKRQn+wkr2pVysDkaEEqtgkE2KlD1Qfyqr5pnN1wDRtkO4syLLlQ9WECRa
-qU848FFWIH9GkJkx7nmRTEd0Ma/WS0O7BnXGxD3fRxNKtc7+PVxvdDb/roq2xiq9
-6ww3f4IvAgMBAAECggEAVsf/FPoCB6cDXDzK7RrHveEFnu5eBXEtHCI3qMS1504b
-bAxMkQkAMa2ICgdKJbiLn5ay1xS15qeslfu7qVIHwKfCKIaG8InHjdw1ZQiZ43GK
-TknonUmKnqhNJfkaxo7qR3hbsC24k5/pS2RVESqtvU8AQXV57TeBXFfzgyfpOOEz
-Nz4c27HfV5AZsDB4GOW1CS7ZdmvZuAfThhTJKh0y1Lqn/7TcVLEStDnFyUZig9gr
-I7WaxltruTgdmmhl5qX0Y1vgwS7APGGLYWTmRRBvyloyqfggCYeU3HDF3qH6RStK
-PsPbbHYW6Xj6LxZSJfVLdAVfFim0yAH1fTAmQXiOkQKBgQDdzLSA5zUBAIR3CTms
-sCn8a0ud8ETPUmTf1RmBMDeIuBTR3va/Q0YVYF8jY+ANrEwut0quIgOwZT4Kmo36
-Sx3F+uE7Gy0XQtDnZk7HeCOsF1fY7r81z7oAh61vFxG8ejSMqjUH8S3tTaCBvGi5
-z2ZjDX2lOsynkBwHu+jvYCbwRQKBgQDyUPWHkO41f6SoTXrEzf77y7Z5Wlim2Ltq
-gLAaEjLPuPsL+G/v7nCw6k6YmRli9f5jn1MGCzaRTxu1hkZlPhPvmfH/AGoSixgB
-CGKMCzOuFT3lQmpr0PZdJBpQ0aAo0R8X48J1X1WuyhZFRmafsrqcYeYPLLrs8aSp
-F51EiOJx4wKBgHZsxjPNP0+5pwCnvFOn+8L6xvUdRKAtrg6VCXkg9YXaQ8OK+NRG
-ZzUir6mFuiuSw1puI0G2h6G3bNjcjNIHKW579aZJbFnvIvtqdMWsL9/C1vTNBOsG
-bpm517Bo8N9hoI06k/YZoBwERp7+qiwLwO9CVMPUEVGmcFDw0/57qE7BAoGAXNwv
-q5sb3n6jAChpAuCada4UqJ652H64G9FjKyqMNFSESIsVgpISt4mjQxFwzB7PNShK
-bceASuFREhh6A7RteWW6emmTPLX3JfYrpdVpIa5cQO5Z0CWNNBZwy0X6JYo2azZK
-QCaftFMXg8a97zdUEjfmLSyMGd9kpbQ5/ZijhtcCgYBpmEOKN1SuGyEQ8gTz/02j
-Xz0XW63Dw8JX8s1Ck5u2uM96gsnHfwuC+iZTSu6FIS1e+3NOCEGQVy10rMzR/TLq
-h13ryz65qQtjaNl1gkD2Xf/ENxML57CU9nK5v+0zb+Qlk3CMKm0N06UmUTUDcE/T
-wIMUuL1LfdQ0b4bnS5JjjA==
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCXDWVfqxLFohyp
+yfp/DS/tphmvt61c9WUAH7o34CmYXTjZlxkdtmhFHQznRW12oyt1Y78PQq0sbngL
+/uukb+3axYRIN4wQSLkAIVvdm6UbP7TnxzkCrGuEB0IEQuMKEGWYgKHwgwVcPovJ
+jsqYPZVvyoixlBInz9FgdjbhlSPsFOSTVvZzK5Vzv6mO9Jff60P2ZoUcoiDrJLUz
+nM6ITQ7+c38dZerOuBFV73N0CIsp4mO25//kqg0JXpmkAgt8TprOV3ngcuTCrdCe
+1SyLY850DO1cQj81HvSzlG6tkhfKKJx2NfKEMGhYl/DKc6er8q1ld2eAdNnMJ5Yr
+3UqQJRvjAgMBAAECggEABCdyfe1hfcfrNowpycFeFmYMAiDGBsQaO/pOD71Sx4WW
+IP3JILUPLF9pJxm/XJTjhr3wgjCY50Y+d5ineFqPqwWQPBWIUjuobXrStkcn2kD9
+Q6aJ8ys0uxPMBVFQQcruUFcGnRZYA7prSRFr5utCpyllwE8h1LB1bKFeTwPf/LJn
+lFHLr34Vmr6whW3dSOn/QyjSRWXx6b88xc32yfniqI8JZrC5ul0J1ulGB0QRhGkB
+tOr2udYIZKjJkgYPQAL644NDEXL5B5iWUvzVzAyNTsLD/uSb4M/KwWxBlM1kvOGU
+GSNr/yKhwqHvV7zpAyjttEltRi44Nt9ULekEVUB+WQKBgQDHgC+vzZRisjhF46rn
+h5OkPMIWEWOAbJ0+fWFDGncDaTOc++QM1WNgJR24zODTKl92EwQHDVAMesPpO614
+DaoKarmvSqcSRDjKvcyij0GTnDe8qPYPn5aJd9vW4vSsqGNUlevClosCxiavSDrV
+Dyt48PejuX+UW3iqkl1QdTiKNwKBgQDB1LK1lcMrQJBY55lpsF2ovvsEDtwOPR93
+Icr3VtMpZTB0R442WzpvAhKaDLLaKUp5pzRGL4tN3pfpD2tz15+neKn4saE8zLoo
+1V0VDoBx9PCvfoP98/zbXZJiVFkdur+B6kz2ImUdnajL2lgnZjJSKGU3+QcEa8be
+vCEbuuI1tQKBgEtK57Rz5BKzrGVEHVYwZtpPS/jEK0dmtpnE/BY1QMF6zM3siw8E
+4fKBT4xVVmyby1dmqdh3f8SV+KZnrXpnvcscqGOtywwrzztFEpwvcNJi3O5I3kTb
+OBc9syyQj0MycfqUclwIMQKlAiiZHAsZJJEKcIvMEIhPVWUaxDj/ZI1VAoGBALuz
+JXwCkwUHeMEfRci7KXeHubysRxziw92xs8LhKYskmk+AG483uNZWw3y6X1g0NBKq
+gEyXLecybxkZ/YwoekrK5o4WNXWTK5LvSzxg1rGD6K+JbHg4Fdy4OFFQLRxFA77j
+DbQsio6rZhJCVquQo5o71Ajn9oupcuOuesFHsFchAoGAYedyPqf8UDgTbsPlA2uo
+kzPanG7Vb5JtHbE14XsAaQP27EX6Ww96oCjEMEhq3KqHj+CQGga0uj6dhgWw+8fl
+s8sLz456U6XkRNjjDJ5luklxvNY2ejETcnZ16t0dp8TtJ8xiFvYO3XyIioCMdtY9
+3TVra0Izy2Wgc1dg4Pz9VKA=
 -----END PRIVATE KEY-----
--- a/tests/auto/external_IODevice/cert/rootCA.pem
+++ b/tests/auto/external_IODevice/cert/rootCA.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDBTCCAe2gAwIBAgIURtkt3RNfea7kp8Dl9UBvr0mXk0kwDQYJKoZIhvcNAQEL
-BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzA1MzEwODExMTRaFw0yNTA5MTMw
-ODExMTRaMBIxEDAOBgNVBAMMB1F0Uk8gQ0EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
-DwAwggEKAoIBAQDR8adUoBxw9L7E5rkMbvNmY2i5aJUecLSogXZ5arctuBeu4PQC
-2YzZU/0ff/kq8n/NhXATH11D7CXad7pYPbMaP1dqPkBdTAkRjkpwlkhGHQv/9teQ
-vMWesVr8rQ4ddk2DK/jmEEyFD3MwTgX8jsw7nhM+mBnhbvK/7DEEoAK6pfDuLl8f
-sVAHdc+3RClO0FQAYt1psvPB8Yvqx2NXy3GKRQn+wkr2pVysDkaEEqtgkE2KlD1Q
-fyqr5pnN1wDRtkO4syLLlQ9WECRaqU848FFWIH9GkJkx7nmRTEd0Ma/WS0O7BnXG
-xD3fRxNKtc7+PVxvdDb/roq2xiq96ww3f4IvAgMBAAGjUzBRMB0GA1UdDgQWBBSP
-+7lx7VdfCz4w2UGzgCsb1e69XzAfBgNVHSMEGDAWgBSP+7lx7VdfCz4w2UGzgCsb
-1e69XzAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDFBHvmndkO
-Up7scewQk8zACOcqyx33kUUT6NiAXDPglzITkVjkNU0r93aek6q5k4WmR2M2nm9n
-ptVRWMacZumxkT0wTqDEyH1cWBEjfNBBtI3JwRc2/6JCIEmqnlBc8/YOZ1xSh5/1
-T6x8dwNkR43wq1IoWilWIU8/trT97mly3X6yIzITzWEsGIiQmjzWvEJOg8tIjIsV
-d7JUDZ+MDC4Rhh/a7v6m7He3dchCsrsIVq/RiJyy8e8mbf0o0slB/SQrxbP0+lQ3
-TlE8Wk/FIpAptx0D5qUnq8E6CWxfUv+R6OK/trCQ+ILZgy2ReeM3yX5AZu7brKI3
-tHlEN2BVbdEo
+MIIDBTCCAe2gAwIBAgIUQExJNJG4q9PKcF6VZhx3N7QwAF0wDQYJKoZIhvcNAQEL
+BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzEyMDMyMTQ0MzdaFw0zMzEyMDEy
+MTQ0MzdaMBIxEDAOBgNVBAMMB1F0Uk8gQ0EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+DwAwggEKAoIBAQCXDWVfqxLFohypyfp/DS/tphmvt61c9WUAH7o34CmYXTjZlxkd
+tmhFHQznRW12oyt1Y78PQq0sbngL/uukb+3axYRIN4wQSLkAIVvdm6UbP7TnxzkC
+rGuEB0IEQuMKEGWYgKHwgwVcPovJjsqYPZVvyoixlBInz9FgdjbhlSPsFOSTVvZz
+K5Vzv6mO9Jff60P2ZoUcoiDrJLUznM6ITQ7+c38dZerOuBFV73N0CIsp4mO25//k
+qg0JXpmkAgt8TprOV3ngcuTCrdCe1SyLY850DO1cQj81HvSzlG6tkhfKKJx2NfKE
+MGhYl/DKc6er8q1ld2eAdNnMJ5Yr3UqQJRvjAgMBAAGjUzBRMB0GA1UdDgQWBBTv
+r1oi6bupMmgta/v+Dek0O9xCcDAfBgNVHSMEGDAWgBTvr1oi6bupMmgta/v+Dek0
+O9xCcDAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQBCWGZURlHB
+KfFfDSI4+4EkNax/LzYPpcI5w6HAQHG7RE5XanihJ4exE2Aqnwzu4UWZ/p7M2mPz
+pVN7NgGkqYN7TQBoRbyKwZO3KGw0x5ARm4SNW/T31I/u7zLAh66RXnMUJ46QaYH6
+hlJ2zusxp7saXgppx/ipJtJKd1IFIuKXMUS8uNEZ1Zly3ZMYt1udh6IYZqMcYbnG
+aEsvxMQq322mH4vGSonCXAnmuCESw/wR4PX+JQei3Z/0VYWhlxBKjyLrL0b9EQmK
+ps3wu1w4uuzWYiLNlDCQ+Ri6ej3APb9jhLg2KJ4SLB7UTvXN5/7CRf+pN182yo0n
+Xv1An8yXYqCH
 -----END CERTIFICATE-----
--- a/tests/auto/external_IODevice/cert/rootCA.srl
+++ b/tests/auto/external_IODevice/cert/rootCA.srl
@@ -1 +1 @@
-7390820BD47F5A3C7398B7C8DCA218B35C2C2E2B
+7390820BD47F5A3C7398B7C8DCA218B35C2C2E2F
--- a/tests/auto/external_IODevice/cert/server.crt
+++ b/tests/auto/external_IODevice/cert/server.crt
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDNTCCAh2gAwIBAgIUc5CCC9R/WjxzmLfI3KIYs1wsLiowDQYJKoZIhvcNAQEL
-BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzA1MzEwODExMTVaFw0yNTA5MDIw
-ODExMTVaMBQxEjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQAD
-ggEPADCCAQoCggEBAJIhl8T4lEioQYGoIFar95zkXi32PrRQ5cK36fP4mVgbJbmV
-ME1y6ZCG/29WMK9Ja3rQn2nkm4RiKVVd4/9WuVM7Eof7w1LRJ5kPxmfUFFc2WSvE
-VqazTikBUUYGW0otL0HEgyPBd39lexktJgk9AshGDOrcRZfNGn1SYlqh3g0V+TCz
-ZiN1drdIU19ZXWYBpzZcPibHxp0oTpDBIgiAD2TenzuLZkQPTuf+7BVchZ0Gy3nL
-VS5E1+M5uKjgaobOTdDLVjrHPgVTH/pa5ZCqELeRFVz6JyioC4HJy2HuZXeZNAQL
-zlqMAWNoUAMsvGKJVvg81ps0BmOrNKOaS7WzkZkCAwEAAaOBgDB+MB8GA1UdIwQY
-MBaAFI/7uXHtV18LPjDZQbOAKxvV7r1fMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgTw
+MIIDNTCCAh2gAwIBAgIUc5CCC9R/WjxzmLfI3KIYs1wsLi4wDQYJKoZIhvcNAQEL
+BQAwEjEQMA4GA1UEAwwHUXRSTyBDQTAeFw0yMzEyMDMyMTQ0MzhaFw0zMzExMzAy
+MTQ0MzhaMBQxEjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQAD
+ggEPADCCAQoCggEBAKXeQUCDijqsGA3hTCwwD0fTa9afzuSrYJn3NzjeEOfNERnx
+iICjkMmHMXIw9s3NYAVDhOAQjXQLXdAWymelZq8tjOyrN3P+K6C6SrMl/bf88pTZ
+h49qdDSwjAcmhM0ZIqFEDfeUSMbV7JCiVa0ELPluKKAKcvUghHuPf9t/PhgHSdKU
+IXcifTX1RZzOBuHwuCOe+3TgxISaGZV/B4bdiOoIleiMHhL8CC07seLEp7gtYhh6
+l3pEpYStb66/6kSdliL9vTE10pR5XEsbu94aNBlEcLGxwHODRo2vfgkIAGQTuzGk
+aD9mlbmGb4o20cBoBmnSEPghR9KNq9Sgh+CemQcCAwEAAaOBgDB+MB8GA1UdIwQY
+MBaAFO+vWiLpu6kyaC1r+/4N6TQ73EJwMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgTw
 MBMGA1UdJQQMMAoGCCsGAQUFBwMBMA8GA1UdEQQIMAaHBH8AAAEwHQYDVR0OBBYE
-FOTMLTCG7GZ2LY1kn5tI7GTxn4R1MA0GCSqGSIb3DQEBCwUAA4IBAQC46Sr1neC2
-g9WikXFnptg6SvrX4xIowxV4f2y2iHSozcBz5lZYLRqLhsl0EuxO9UuCbqPx9nqi
-wHBk84S7METP+tmdvDBj7TtLEml7qwQIcR58YLD5Cu3oQLO1jcupq5DFAbKPBTBu
-ybJq+osofibogH6qdbYLyTjjFcTkg9A9VHvWEVhfaT355GmR4Yt+RKL9GiapDYF5
-ZiB8PRsX1HQzccmZsZrFVWb0klLq1tlDjSK1nx+xDBx7qbDLzUK49ie3k5MTVOfr
-E45yOZ6Sh8mHL+gCKcxFk83Wqy8oudChcGEiTEqKxDD268BvB00ECnhF3xAgtM+m
-LZ7CbX52p3bW
+FPkriomNYzq4cL13ZRpyrbYrHVJ2MA0GCSqGSIb3DQEBCwUAA4IBAQAGUGx/5Iiu
+9wDAASfT0xIp03sXkLjjE1Q+h2EYx/+4IoqeyziL5/xkEmqP7aROvgzwya+/z1Gw
+ycctqNPtKGHi6U4jcQQvl6OP9Ij9qae8yTtXufKzA/JJDh5YdOOgV2FqENKeANnd
+QlKoiXXfdtMQPAsK46UaBdL3pjprictUOgy1h7ItaqiPQsaeQ0e6CnjJ7LgOSiff
+oXHwTb9uRP6HuIuie+mG3ow7Q9BVzklHCRW4E1XeD1FiHBcKYDbEQ3TjFzsxd27T
+O3aivrczV79Km/InfaozssOT3cyYZc/BjKEwJZ7vECpmWZ0iYi0qzq5jF8qGLEDf
+1AqKTA1bN11Y
 -----END CERTIFICATE-----
--- a/tests/auto/external_IODevice/cert/server.key
+++ b/tests/auto/external_IODevice/cert/server.key
@@ -1,28 +1,28 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCSIZfE+JRIqEGB
-qCBWq/ec5F4t9j60UOXCt+nz+JlYGyW5lTBNcumQhv9vVjCvSWt60J9p5JuEYilV
-XeP/VrlTOxKH+8NS0SeZD8Zn1BRXNlkrxFams04pAVFGBltKLS9BxIMjwXd/ZXsZ
-LSYJPQLIRgzq3EWXzRp9UmJaod4NFfkws2YjdXa3SFNfWV1mAac2XD4mx8adKE6Q
-wSIIgA9k3p87i2ZED07n/uwVXIWdBst5y1UuRNfjObio4GqGzk3Qy1Y6xz4FUx/6
-WuWQqhC3kRVc+icoqAuBycth7mV3mTQEC85ajAFjaFADLLxiiVb4PNabNAZjqzSj
-mku1s5GZAgMBAAECggEAC0J64QK9XD7Ys06a1UrRXL2s8xItATRVi9Qjevi+Cg+n
-b74K14Dh9dCtGG1V+X+pADDYg66JfCEbKbh/E6LhS0Q7OI6h3Xz/tdfFFCV/frvR
-kaGCgrxAqsCu/DwLpzLFmCX169ihpkeRvV4g9ufEUd2x65vW4ew8HQluyih8RVo8
-+spiAou5hPTT6rSX4A2ywthqyAdeq7hTEyIj6/Ai3gaSrL9htSJn4DxxYq5GiLjN
-WjhFUQWvz45fehLIRRA0yU+wUonGlI75KYasEd1O0cKttAePK0yEtaBuJbYCckYb
-PPD+iFU3KnBILNXVolBSlKOspAOtqDYDpKfSTdImgQKBgQC2lcFjO8lzzTy1Cw1Y
-9TMUR6yzjdQBvLpFVsB8kRQ/13bs/8FYiBhm9RbXE3SXCQ5Np6/FWlRAaGY8ieyg
-5+4B3+TqJvTfPrtS8tYBh8koai/Gc0Rq9jn95KrTe4pcMG+thodto84I/fwMzZHq
-Ns27/ejEjhdqviiGkWjJw3vwQQKBgQDM439pU4pLpzlDeyrSAPgAa4B6D48+BZpE
-Ap6Pb28Dsn3twl7Y8leodUeJIc+JVs1VRpzO8ROxtxGI2STebSHhBe1mEjhJpOPv
-OTwJTocdwd2xEwnxbz4OhNfnlN1ESTM1JZYl3XLANHqdxWb8fn7dYNjncdciOTqM
-LCa8obpLWQKBgCDJyd3BKc8i5yj34CC+f7q/mFLyQEVDmAvFpXF5ODKyxcePsi/s
-2Lz8fGqJMuHAnZHycA03yHN8jAeJGTZYEBTEGT63WjgQ8uDVfPrvYH9iekvf/URc
-DVP70XWC9PjTNerqgr18sg5JVg8Sh4aQ6uc8lKVfgvMHTQ3FZRT+hu5BAoGBALjK
-8y6nF1OWlnJDk2JnV0bhrrXodLRmzzVJ+4K46bK2+RdJOMyI/wDEDmojoZBoRrgE
-AQgwrSebSuhXmQY3/dd7xhtKkZPycVZhE8H9HqZ7JR1BY7VrNNhZfnWzcUHWl6X2
-qY6rRYUPkAvecD2JRZkbcWaF6ZhHb4PfimyVnXW5AoGAd8oiLmBV4AnBfv2YCDY4
-2Ec2+vcXrv4uS4xQq8KM7VLKKKACqOfmgMg//IcxF3EcXayigY711Gb60LVj5zX+
-bpl4ryS+pgH8qtMzGuWiNnUIBCQLp95vhz7WPDj2OmuRA6o+UiUg4bH146Kmq1+u
-mGYP0GQo5eyxWfXL0W5/fJY=
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCl3kFAg4o6rBgN
+4UwsMA9H02vWn87kq2CZ9zc43hDnzREZ8YiAo5DJhzFyMPbNzWAFQ4TgEI10C13Q
+FspnpWavLYzsqzdz/iugukqzJf23/PKU2YePanQ0sIwHJoTNGSKhRA33lEjG1eyQ
+olWtBCz5biigCnL1IIR7j3/bfz4YB0nSlCF3In019UWczgbh8Lgjnvt04MSEmhmV
+fweG3YjqCJXojB4S/AgtO7HixKe4LWIYepd6RKWErW+uv+pEnZYi/b0xNdKUeVxL
+G7veGjQZRHCxscBzg0aNr34JCABkE7sxpGg/ZpW5hm+KNtHAaAZp0hD4IUfSjavU
+oIfgnpkHAgMBAAECggEAGDdWNH/f5F1awlqvHyZscgftJG++1NuBJjQVz7YlfAo6
+9xwwhINyMJIN3IZHNNh0A+C4hhQJ6t7RSqpWe60GIbOXRw208g4yOsGSsAwZuk0c
+8jFoyBNP4EoZmjOmoOEMvBmuUQPztMqSbV1d/9LDvtjf59m5WzjCTt3ZBL1TiKNP
+NGX2NqXBwoKTLrHTWeMku63HOBHzDCTiRfqeCGaG/+i8/FHcz33ZBBuj9ZE6Pz5p
+6TnHgh0jU574vZmAEaWUkPUulO78onBOxjQgpGksbVJFdc5Ls6WRhJAkWJa8ozEM
+u1xzRyJu/O+r3YUXLzjBaB1KrJV1+0n06PUF1YfGrQKBgQDnED4C+LR8hAWuuQa/
+1DlW+1nmkYueMOa9DNIEoyx85yqwvZ/I6z/wEWT27j63ezWUJ5yumWNj7yjg6fze
+/FxtqdxMV8qEh7IgKrF0evgdUQ03naU9VMHzGFzvHgZE5D2a36mjbUG1ycdn1DXH
++l35WbnWE/c4qHnxQ3BunP7jSwKBgQC3xNDUZ8A3MErp7UNT4tGUKGdAPqYRd/gq
+Rw6EOpv782Qu6/sRIQvOo79ctCmelgBLjuNSRnUQPqFwesbcco0BAOTqJEmuMoga
+sy4UMt3t9T1egc0yA0G9ETQ6oL2STEhfsVgh6x52q8Gql7azqFDNXcF6FhvIkpAe
+QEgJdrOPtQKBgHdJ0VnjX/pwhiZ/61YZkEixluYL+dJ/WH/9IGe4ff01srpwCw/F
+M8sSj9dThBlhiHWmRaN2GPO8wpPq+Nt92ZHcAZmAMVNeYr1sjRSfb3xjHbm6r1YM
+bdy8lS31iDMEyjEsiv2OKEBbnZCi4+kL/nv+sIwu3pDnmZzOvjq83nIlAoGAFI50
+mb39V28UgjRjOqf9r6gRXLNlWkDkRr5qFkqEFlv0kfcdv53Ft1jD2DfjC/VVlR4n
+uZVeFPIclvt08UXzFfCmqrZic5T9epceiW3jO3Iaak70yCprGR4qjIBclFfqJKIu
+7KRjZPzzjgSokFI8r6xULzb35qjWbDvekwhvL50CgYBOQBuRSPD44sIkdpKq/Q8q
+51Vn/ggrHr2jNML+vKuyDfuPnlV3RiKtJ6dWt6omTxS8in8GJGpDgFG6eICuhVox
+vuOkm8bXjMZX37Hdog4iNG2xtUImrL2MTeVvGKWxFGR52/3YiqZ6jThA0A1RBCeL
+os0lSz56rZtnpMyG9Gi50w==
 -----END PRIVATE KEY-----
